<!DOCTYPE html>
<html lang="en-us">

  <head>
  
    <meta charset="UTF-8">
    <title>Parvovirus-GLUE by giffordlabcvr: An example-based tutorial - parvoviruses and the mink fur industry</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Parvovirus-GLUE</h1>
      <h2 class="project-tagline">A sequence data resource for parvoviruses (family <i>Parvoviridae</i>).</h2>
      <a href="../../index.html" class="btn">Home</a>
      <a href="./viruses.html" class="btn">Viruses</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="./highlights.html" class="btn">Highlights</a>      
      
      <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>

    <section class="main-content">


	   <h3>
		 <a id="sequenceData" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>How to use Parvovirus-GLUE - an example-driven tutorial</strong>
	   </h3>
       <hr>


	   <p>
         
         This tutorial focuses on <b>carnivore amdoparvovirus 1</b> - also known as
         
         <a target="_blank" href="https://en.wikipedia.org/wiki/Aleutian_disease"><b>Aleutian mink disease virus (AMDV)</b></a>.
         
         It comprises the following steps:
		 <ol>
		   <li> <b>Downloading</b> selected sequences from GenBank.</li>
		   <li> <b>Extracting</b> isolate data from GenBank files.</li>
		   <li> <b>Filtering</b> sequence data to obtain a valid analysis set.</li>
		   <li> <b>Defining virus reference sequences</b> and associating them with annotations.</li>
		   <li> <b>Constructing an alignment</b> using a codon-aware method.</li>
		   <li> <b>Mapping feature coverage</b> of sequence members within an alignment.</li>
		   <li> Reconstructing annotated, midpoint-rooted <b>phylogenetic trees</b>.</li>
		   <li> Using GLUE projects as a basis for <b>collaborative research and resource development</b>.</li>
		 </ol>


 	   </p>
 	   

	   <p>
         If you haven't decided yet whether or not you want to use GLUE, 
         you might find it useful simply to look through this tutorial, as it should
         provide useful insight into the kind of things that can be done with GLUE projects.
 	   </p>

	   <p>
         If you've already committed to using GLUE, you 
         may find it useful to also investigate the
         <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/exampleProject"><b>GLUE example project</b></a>
         when attempting this tutorial.
 	   </p>


       <br>
	   <h3>
		 <a id="Background" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Background 1: Carnivores, parvoviruses and the fur industry</strong>
	   </h3>
       <hr>

        <p><img src="../assets/images/fur-trade-parvoviruses-header.png" alt="Parvoviruses and the fur industry" /></p>


<blockquote>
		 <p>
		   <b>Left to right</b>:
		   stoat/ermine (<i>Mustela ermina</i>);
		   Leonardo da Vinci's '<a target="blank" href="https://en.wikipedia.org/wiki/Lady_with_an_Ermine">Lady with an Ermine</a>' (1489–1491);
		   North American mink (<i>Neovison vison</i>);
		   Women posing with mink furs in the 1930s;
		 </p>
</blockquote>
		 


	   <p>
        Humans have established relatively close relationships with several carnivore species.
        But whereas some are kept as companion animals, others are hunted or farmed for meat or fur.
        In Europe, several 
		 <a target="_blank" href="https://en.wikipedia.org/wiki/Mustelidae">mustelid</a>      
        species have historically been hunted for fur, including
        European mink (<i>Mustela lutreola</i>),
		 <a target="_blank" href="https://en.wikipedia.org/wiki/Lady_with_an_Ermine">ermine</a>      
        (<i>Mustela ermina</i>)
        and European polecat (<i>Mustela putorius</i>).
        However, the fur of the    
		 <a target="_blank" href="https://animaldiversity.org/accounts/Neovison_vison/">North American mink </a>      
        (<i>Neovison vison</i>) is considered
        superior in quality to all of these.
 	   </p>

	   <p>
         Furthermore, while records of efforts to breed ermine (stoats) in captivity exist,
         these ventures have apparently been short-lived. By contrast, North American mink
         are extensively farmed. 
         The first mink farms were founded in the 1860s, in Upstate New York, and farming
         of American mink was introduced into Europe in the early 1930s. The breeding of
         various fur colour mutants led to a boom in the mink industry in the two decades
         that followed.  
 	   </p>

	   <p>
         One unintended consequence of the post-war boom in mink farming was the
         introduction of American mink into Europe (as escaped animals took up residence
         in local habitats). American mink (hereafter referred to simply as ‘mink’)
         are now a relatively widespread invasive species in Europe.         
 	   </p>

	   <p>

         In the 1930s a severe disease emerged in farmed mink. This disease was
         originally identified in the Aleutian mink breed and was consequently named
         <b>Aleutian disease (AD)</b>. However, it was soon discovered to afflict American mink
         in general.
         AD is caused by an parvovirus in genus (<i>Amdoparvovirus</i>)
         called <b>Aleutian mink disease virus (AMDV)</b>. This virus was originally considered
         a species in its own right, but has recently been reclassified as a sublineage of
         carnivore amdoparvovirus 1.
      
         AD is presently considered the most important infectious disease
         affecting farm-raised mink.
  	   </p>

	   <p>
 	   </p>


       <br>
	   <h3>
		 <a id="Background" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Background 2: AMDV-related parvoviruses in wild versus farmed carnivores</strong>
	   </h3>
       <hr>
		 
	   <p>
         Infection with AMDV - or related amdoparvoviruses -
         is apparently widespread in wild mink as well as in farmed
         animals. In addition, related viruses have been identified in several other carnivore
         species, including gray foxes, skunks, raccoon dogs and red pandas. However,
         relatively little is known about the biology of amdoparvovirus infection in the
         natural environment, or the broader distribution of amdoparvovirus infections
         in wild species.
 	   </p>

	   <p>


        <p><img src="../assets/images/fur-trade-parvoviruses-midlevel.png" alt="Carnivores, parvoviruses and the fur trade" /></p>


<blockquote>
		 <p>
		   <b>Left to right</b>:
		   gray fox (<i>Urocyon cinereoargenteus</i>);
		   striped skunk (<i>Mephitis mephitis</i>);
		   raccoon dog (<i>Nyctereutes procyonoides</i>);
		   red panda (<i>Ailurus fulgens</i>);
		 </p>
</blockquote>
		 

         <u>Most importantly</u>, it is not clear whether the pathology of AD in captive mink is
         typical of disease that occurs in the wild, or if factors associated with fur
         farming somehow enabled the emergence of the disease. 
 	   </p>


	   <p>
         Increased availability
         of molecular sequence data mean it might now be feasible to gain some insight
         into the natural history and evolution of amdoparvoviruses, and this in turn
         may shed light on the emergence of AD.
 	   </p>


	   <p>
         In this tutorial we will use the <b>Parvovirus-GLUE</b> project
         and published parvovirus sequence data to <b>(i)</b>
         investigate AMDV distribution diversity and evolution, and <b>(ii)</b> establish
         a sequence-based resource for AMDV.
 	   </p>


       <br>
	   <h3>
		 <a id="DownloadFromNcbi" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>1. Downloading all available AMDV sequences from GenBank</strong>
	   </h3>
       <hr>



	   <p>
         To download all AMDV entries in NCBI GenBank, we 
         will use a customised version of GLUE's 'ncbiImporter'
         <!-- <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/moduleReference/moduleType/ncbiImporter">'<b>ncbiImporter</b>'</a> -->      
         module. Our project-specific configuration of the module can be viewed
		 <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/modules/build/genus/amdo/amdoNcbiImporterExampleAmdv.xml"><b>here</b></a>.
         Viewing the file, you can probably see that it is configured to download sequences
         based on a query phrase:

	   <p>
        <i>"Carnivore amdoparvovirus 1"[Organism] AND 200:2000[SLEN]</i>
 	   </p>


	   <p>        
         This 'eSearchTerm' is a standard NCBI entrez text query that
         specifies all GenBank entries labelled "beak and feather disease" in the
         'Organism' field and between 200-5000 nucleotides (nt) in length.

 	   </p>

	   <p>
         To use the module, first initiate GLUE on the command line as follows: 
         
	   
<pre>
<code>MyComputer:Parvovirus-GLUE rob$ gluetools.sh 
GLUE Version 1.1.103
Copyright (C) 2015-2020 The University of Glasgow
This program comes with ABSOLUTELY NO WARRANTY. This is free software, and you
are welcome to redistribute it under certain conditions. For details see
GNU Affero General Public License v3: http://www.gnu.org/licenses/

Mode path: /
GLUE></code></pre>


          
         Notice from the first line that I'm in the 'Parvovirus-GLUE' directory when
         I initiate GLUE.
         This means that, by default, it will be my 'working directory' - when I reference
         a file from the GLUE console, I'll need to do so <u>relative to this directory</u>.
 	   </p>

	   <p>
 
         Next, navigate to the 'parvoviridae' project as shown here:
	   </p>

<pre>
<code>GLUE> project parvoviridae 
OK
Mode path: /project/parvoviridae/</code></pre>

 	   </p>
 
	   <p>
         Now create the module using it's
		 <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/modules/build/genus/amdo/amdoNcbiImporterExample.xml"><b> configuration file</b></a>,
         which is contained in the Parvovirus-GLUE project, as shown here:

<pre>
<code>Mode path: /project/parvoviridae/
GLUE> create module -f modules/build/genus/amdo/amdoNcbiImporterExample.xml
OK
(1 Module created) </code></pre>

 	   </p>

 	   <p>
         To run the module, execute the following command in the GLUE shell:

<pre>
<code>Mode path: /project/parvoviridae/
GLUE> module amdoNcbiImporterExample import</code></pre>

 	   </p>
 	   <p>

         When I ran this command on the 2nd March 2022, I obtained the following output:


  <pre>
  <code>ncbiImporterSummaryResult
  totalMatching: 1777
  present: 0
  surplus: 0
  missing: 1777
  deleted: 0
  downloaded: 1777</code></pre>

         Since the GenBank database is continually expanding, you may find
         that you obtain more sequences than this when running the same 
         command at a later date. Note that next time the module is run, only 
         the missing sequences will be downloaded. 
	   </p>
 
 	   <p>
 	     Now export the sequences to a 'source' directory as follows.
	  

<pre>
<code>Mode path: /project/parvoviridae
GLUE> export source ncbi-curated-parvo-amdv -p sources/genus/amdo/</code></pre>
		</p>

  
       <br>
 	   <h3>
		 <a id="ParseGenBankXML" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>2. Importing AMDV sequences and metadata from GenBank files</strong>.
	   </h3>
       <hr>


	   <p>
         Now that we've downloaded a set of AMDV sequences, we can incorporate them into 
         the Parvovirus-GLUE project.
         Load the sequences as follows:

<pre>
<code>Mode path: /project/parvoviridae
GLUE> import source sources/genus/amdo/ncbi-curated-amdv/</code></pre>

 	   </p>
 
 
	   <p>
         The next step is to incorporate additional, sequence related data.
         To do this it we need to consider the underlying schema of the database
         underlying Parvovirus-GLUE, which is defined in 
		 <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/glue/build/core/parvoSchemaExtensions.glue"><b>this file</b></a>.
 	   </p>


	   <p><img src="../assets/images/schema-extensions.png" alt="DB schema" /></p>


<blockquote>
	   <p>
		 <b>Parvovirus-GLUE schema extensions</b>.  All GLUE projects have a 'sequence' table.
		 In Parvovirus-GLUE we've extended this table to capture taxonomic information as 
		 well as some other sequence-associated data fields. In addition, we've defined 
		 a separate 'isolate' table that is linked to the 'sequence' table via the
		 'sequenceID' field (always unique to every sequence). The isolate table contains
		 information specific to the isolate that the sequence was obtained from - e.g.
		 host species from which the isolate was obtained, isolate name, and spatiotemporal
		 data associated with isolation.

	   </p>
</blockquote>
 
	   <br>


	   <p>
          All AMDV sequences have the same taxonomic information.
          This data can be entered on a sequence-by-sequence basis using GLUE commands, but
          to make the process more efficient, we can instead use 

		  <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/glue/tutorial/exampleSetTaxonomicDataAmdv.js"><b>a script</b></a>.
	   </p>
 	  


<pre>
<code>Mode path: /project/parvoviridae
GLUE> run script glue/tutorial/exampleSetTaxonomicDataAmdv.js</code></pre>



	   <p>
          Before we can import sequence-associated data to the isolate table we need to first
          create a link to that table for each sequence we have imported.
          This can be accomplished on a sequence-by-sequence basis using GLUE commands.
          Alternatively, to make the process more efficient, we can use a script.
	   </p>
 	   
	   <p>
		  The script is run as follows:

 <pre>
 <code>Mode path: /project/parvoviridae
 GLUE> run script glue/tutorial/exampleLinkIsolateDataAmdv.js</code></pre>
	  



	   <p>

		 An 
         <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/modules/build/core/parvoGenbankXmlPopulator.xml"><b>appropriately configured genbankPopulator module</b></a>   
		 can now be used to extract sequence and isolate information from GenBank files.
		 We can use this module to extract useful isolate-related information 
	     that is embedded in the "notes" section of the GenBank file.
	   </p>

 <pre>
 <code>Mode path: /project/cress
 GLUE> module parvoGenbankXmlPopulator populate</code></pre>


	   <p>
		 We can now inspect the data in the isolate table via the command line, to
		 see if it has been extracted as expected:
	   </p>
		 

<pre>
<code>Mode path: /project/parvoviridae
GLUE> list sequence sequenceID isolate.isolate isolate.country -w "name = 'AMDV'"

+============+========+============================+===================================+
| sequenceID | length |   isolate.host_sci_name    |         isolate.country           |
+============+========+============================+===================================+
| AF205380   | 690    | Mustela lutreola           | Spain                             |
| AF205381   | 687    | Neovison vison             | Spain                             |
| AF205382   | 690    | Lutra lutra                | Spain                             |
| EU652446   | 782    | Neovison vison             | China                             |
| EU652447   | 782    | Neovison vison             | China                             |
| EU652448   | 785    | Neovison vison             | China                             |
| EU652449   | 785    | Neovison vison             | China                             |</code></pre>		   

	   <p>
		 
		 Note that a
         <a target="_blank" href="https://www.w3schools.com/sql/sql_where.asp"><b>'where clause'</b></a>         
		 is used to limit the query to AMDV. Where clauses can be used in GLUE to 
		 control how data are selected, and can reference any data field represented 
		 in the underlying <b>project database schema</b>. The schema can be extended with new 
		 fields and tables as required.

	   </p>


  
       <br>
 	   <h3>
		 <a id="CreateMsa" class="anchor" href="#CreateMsa" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>3. Creating an alignment of all AMDV sequences</strong>.
	   </h3>
       <hr>


   
		 <p>
		   To create an alignment, we need to first create a <b>constrained alignment</b> object as follows:
		 </p>


<pre>
<code>Mode path: /project/parvoviridae
GLUE> create alignment AL_AMDV -r REF_MASTER_Amdo_AMDV
OK</code></pre>

  
		 <p>		   
		   We can now specify which sequences belong to this constrained aignment object, as follows:
		 </p>


<pre>
<code>Mode path: /project/parvoviridae
GLUE> alignment AL_AMDV add member --whereClause "source.name = 'ncbi-curated-amdv'"
OK</code></pre>

		 <p>
		   Now we can construct the alignment.
		   The Parvovirus-GLUE project contains a 
		   <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/modules/build/genus/amdo/parvovirusCompoundAligner.xml"><b>parvovirus-specific configuration file</b></a>
		   for GLUE's  'compoundAligner' module.
		 </p>

<pre>
<code>GLUE> compute alignment AL_AMDV parvoCompoundAligner</code></pre>


		 <p>
		   Now let's inspect the resulting alignment:
		 </p>

<pre>
<code>GLUE> alignment AL_AMDV show statistics
OK
GLUE> project parvoviridae alignment AL_AMDV show statistics
+======================+=====================+============================+=========================+==========+==========+=============+=============+
| sequence.source.name | sequence.sequenceID | referenceNtCoveragePercent | memberNtCoveragePercent | minRefNt | maxRefNt | minMemberNt | maxMemberNt |
+======================+=====================+============================+=========================+==========+==========+=============+=============+
| ncbi-curated-amdv    | AB044558            | 7.6025827952509895         | 100.0                   | 3043     | 3407     | 1           | 365         |
| ncbi-curated-amdv    | AB044559            | 7.6025827952509895         | 100.0                   | 3043     | 3407     | 1           | 365         |
| ncbi-curated-amdv    | AF107626            | 6.998541970422829          | 100.0                   | 587      | 922      | 1           | 336         |
| ncbi-curated-amdv    | AF107627            | 6.998541970422829          | 100.0                   | 587      | 922      | 1           | 336         |
| ncbi-curated-amdv    | AF107628            | 6.998541970422829          | 100.0                   | 587      | 922      | 1           | 336         |
| ncbi-curated-amdv    | AF107629            | 6.998541970422829          | 100.0                   | 587      | 922      | 1           | 336         |
| ncbi-curated-amdv    | AF107630            | 6.998541970422829          | 100.0                   | 587      | 922      | 1           | 336         |
| ncbi-curated-amdv    | AF107631            | 6.998541970422829          | 100.0                   | 587      | 922      | 1           | 336         |
| ncbi-curated-amdv    | AF107632            | 6.998541970422829          | 100.0                   | 587      | 922      | 1           | 336         |</code></pre>


		 <p>
		   (only the first few results are shown here)
		 </p>



		 <p>
		   Or alternatively, list alignment members, along with their associated data, like this:


<pre>
<code>GLUE> alignment AL_AMDV 
OK
Mode path: /project/parvoviridae/alignment/AL_AMDV
GLUE> list member sequence.sequenceID sequence.name sequence.isolate.isolate
+=====================+===============+====================================+
| sequence.sequenceID | sequence.name |      sequence.isolate.isolate      |
+=====================+===============+====================================+
| AB182568            | AMDV          | YCC-IN1P                           |
| AB182569            | AMDV          | YCC-IN2P                           |
| AB182570            | AMDV          | RRP-JP11P                          |
| AB182571            | AMDV          | RRP-JP12P                          |
| AB182572            | AMDV          | BTP-SA11P                          |</code></pre>


 		 </p>

		 <p>
		   (only the first few results are shown here)
		 </p>



       <br>
	   <h3>
		 <a id="MapCoverage" class="anchor" href="#MapCoverage" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>6. Mapping feature coverage in AMDV sequences</strong>.
	   </h3>
       <hr>

	   
		 <p>
		   Now that we have created an alignment of AMDV sequences that is constrained to 
		   the AMDV reference sequence, we can use this alignment in combination with GLUE's
           featurePresenceRecorder module to examine feature coverage within individual AMDV
           sequences.

 		 </p>

		 <p>
           
		   Run the module as follows to generate feature coverage data for the alignment:
		 </p>

<pre>
<code>Mode path: /project/parvoviridae
GLUE> module parvoFeaturePresenceRecorder record feature-presence AL_AMDV -f whole_genome -d

+=======================+=============================+============================+===================================+=========================+=====================+
| member.alignment.name | member.sequence.source.name | member.sequence.sequenceID | featureLoc.referenceSequence.name | featureLoc.feature.name | referenceNtCoverage |
+=======================+=============================+============================+===================================+=========================+=====================+
| AL_AMDV               | ncbi-curated-amdv           | AB044558                   | REF_MASTER_Amdo_AMDV              | whole_genome            | 7.6025827952509895  |
| AL_AMDV               | ncbi-curated-amdv           | AB044558                   | REF_MASTER_Amdo_AMDV              | VP1                     | 18.775720164609055  |
| AL_AMDV               | ncbi-curated-amdv           | AB044559                   | REF_MASTER_Amdo_AMDV              | whole_genome            | 7.6025827952509895  |
| AL_AMDV               | ncbi-curated-amdv           | AB044559                   | REF_MASTER_Amdo_AMDV              | VP1                     | 18.775720164609055  |
| AL_AMDV               | ncbi-curated-amdv           | AF107626                   | REF_MASTER_Amdo_AMDV              | whole_genome            | 6.998541970422829   |
| AL_AMDV               | ncbi-curated-amdv           | AF107626                   | REF_MASTER_Amdo_AMDV              | Rep78                   | 18.95093062605753   |
| AL_AMDV               | ncbi-curated-amdv           | AF107627                   | REF_MASTER_Amdo_AMDV              | whole_genome            | 6.998541970422829   |</code></pre>


	   
		<p>

		 To inspect the alignment that is selected using the where clause above, GLUE's
		 fastaAlignmentExporter module can be used.
		 
	   </p>


<pre>
<code>Mode path: /project/parvoviridae
GLUE> module fastaAlignmentExporter export AL_AMDV -r REF_MASTER_Amdo_AMDV -f Rep78 -w "fLocNotes.featureLoc.feature.name = 'Rep78' and fLocNotes.ref_nt_coverage_pct >= 80" -p</code></pre>

  
		<p>
		  The exported alignment can be found 
		  <a target="_blank" href="../../trees/tutorial/AMDV-REP78-80pct-coverage.aln.fna"><b>here</b></a>.

	   </p>


       <br>
	   <h3>
		 <a id="buildPhylogenies" class="anchor" href="#buildPhylogenies" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>7. Reconstructing phylogenetic trees</strong>.
	   </h3>
       <hr>
	   
	   <p>
         In the final step in this tutorial, we will reconstruct a phylogeny of AMDV
         Rep78 genes using the feature coverage tables we have just generated.


<pre>
<code>Mode path: /project/parvoviridae
GLUE> module raxmlPhylogenyGenerator generate nucleotide phylogeny AL_AMDV -r REF_MASTER_Amdo_AMDV -f Rep78 -w "fLocNotes.featureLoc.feature.name = 'Rep78' and fLocNotes.ref_nt_coverage_pct >= 50" -o trees/example/amdv-all-genbank-rep.tre NEWICK_BOOTSTRAPS
</code></pre>

		 Before you attempt this, bear in mind that reconstructing this phylogeny will
		 take some time (e.g. around an hour).  
		 
	   </p>

	   
		<p>
		  Export annotations for the trees as follows:


		</p>	 
  

 
<pre>
<code>Mode path: /project/parvoviridae
GLUE> module parvoFigTreeAnnotationExporter export figtree-annotation AL_AMDV -w "fLocNotes.featureLoc.feature.name = 'Rep78' and fLocNotes.ref_nt_coverage_pct >= 80" -f tutorial/amdv-annotations.tsv</code></pre>

 
		<p>
		  The finished trees with annotations can be found 
		  <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master/trees/tutorial/amdv-all-genbank-rep.tre.pdf"><b>here</b></a>.
          One way to view these trees is by using Andrew Rambaut's
		  <a target="_blank" href="https://github.com/rambaut/figtree/releases"><b>FigTree</b></a>
		  program.
		</p>





       <br>
	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>
       <hr>
 	   
		<p>

        Canuti M, McDonald E, Graham SM,  Rodrigues B,  Bouchard E,  Neville R, Pitcher M,  Whitney HG, and HD Marshall
  		<b>(2020)</b>
		<br>
         Multi-host dispersal of known and novel carnivore amdoparvoviruses
		<br>
		<i>Virus Evolution</i>
		<a href="https://academic.oup.com/ve/article/6/2/veaa072/5912945" target="_blank">[view]</a> 
		<br>
		<br>


		Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		<b>(2018)</b>
		<br>
		GLUE: A flexible software system for virus sequence data.
		<br>
		<i>BMC Bioinformatics</i>
		<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		<br>
		<br>

		</p>



	  <footer class="site-footer">
	 
		<span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/">
		  Parvovirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.
		</span>

		<span class="site-footer-credits">
		  This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.
		</span>
	   
	  </footer>

    </section>


  </body>
  
</html>



