<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Parvovirus-GLUE by giffordlabcvr: Highlights</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>


  <body>
    <section class="page-header">
      <h1 class="project-name">Parvovirus-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of parvoviruses.</h2>

      <a href="../../index.html" class="btn">Home</a>
      <a href="./viruses.html" class="btn">Viruses</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <!-- <a href="./tutorial.html" class="btn">Tutorial</a>-->
      <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master" class="btn">GitHub</a>
    </section>



    <section class="main-content">


	   <h3>
		 <a id="'Highlights" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Parvovirus-GLUE: Highlights</strong>
	   </h3>
	   <hr>

	   <p>
		 These highlights pages aim to provide a brief overview of selected data items
		 contained within the <b>Parvovirus-GLUE</b> project. 
	   </p>


	   <h3>
		 <a id="Dependo.54.Cavia" class="anchor" href="#Dependo.54.Cavia" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>EPVs: Dependo.54.Cavia (enRep)</strong>
	   </h3>
	   <hr>


		<p>
          
          Dependo.54.Cavia (also called enRep) is one of several 
          <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/tabular/eve/parvovirinae/epv-dependo-refseqs-side-data.tsv"><b>dependoparvovirus-derived EPVs</b></a>
          identified in the
          germline of guinea pigs (genus Cavia). 
          We identified two groups of elements that spanned the rep gene.
          The first includes enRep sequences from both guinea pig species examined.
          The second includes a longer element that spans an entire rep gene. This second
          group of elements is much older, being shared across rodents species that
          diverged >70 million years ago.
          
        </p>



       <br>


	   <p><img src="../assets/images/epv-dependo.54-cavia.jpg" alt="EPV-Dependo.54-Cavia and Myosin 9 homolog" /></p>


<blockquote>
		  left to right: Guinea pig, Skeletal muscle fibres, parvovirus virion.       
</blockquote>



       <br>


 
		<p>
          While most of this EPV sequence is degraded, the portions included in the
          enRep-Myo9 gene are intact in multiple species of guinea pig (genus Cavia),
          consistent with evolution under purifying selection.
        </p>

 
		<p>
          The broad expression of enRep-Myo9 mRNA, the conservation of it’s EPV-derived
          regions in multiple species of guinea pig (genus Cavia), indicate that this
          host-virus fusion gene encodes a protein with a physiologically relevant role.
        </p>

          
		<p>
          The viral portions of enRep-Myo9 derive from an ancient dependoparvovirus
          (genus Dependoparvovirus)  that was incorporated into the genome of caviomorph
          rodents >6 million years ago.
          
        </p>

		<p>
           Related data items: 
		  <ol>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/dependo/fasta-refseqs-dependo-epv/dependo.54-cavia.fna">EPV-Dependo.54-Cavia sequence</a></li>
			<!--
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/tabular/eve/parvovirinae/epv-erythyro-refseqs-side-data.tsv">Fusion product (AA)</a></li>
			-->
		  </ol>

  		</p>










	   <h3>
		 <a id="Hamaparvoviruses" class="anchor" href="#Dependo.54.Cavia" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Viruses: Hamaparvoviruses</strong>
	   </h3>
	   <hr>


		<p>

          Occasionally, our investigations of WGS databases turn up sequences that do not derive from
          endogenous parvoviral elements (EPVs), but instead from infectious viruses
          that have contaminated genomic DNA samples.
          In 2017 we reported sequences derived from viruses belonging to genus <i>Chaphamaparvovirus</i>
          in WGS data of diverse vertebrate species.
          Detection and analysis of these sequences indicated that the host range of
          'chappaparvoviruses' (as the group was then known) encompassed a diverse range of
          vertebrate species.

		</p>

           
        <p>
          Chaphamaparvoviruses are representatives of a newly described parvovirus subfamily: ''<i>Hamaparvovirinae</i>'.
          Although relatively little is known about these viruses (most have only been described at sequence-level) 
          it is becoming clear that they are very widely distributed among vertebrate species,
          and that some are associated with disease.
          For example, porcine parvovirus 7 (PPV) is one of the organisms associated with 
          Stillbirths Mummification Embryonic Death and Infertility (SMEDI) syndrome in domestic pigs, while
          <a target="_blank" href="https://www.jax.org/news-and-insights/jax-blog/2019/august/mouse-kidney-parvovirus">mouse kidney parvovirus</a>
          is associated with “inclusion body nephritis/nephropathy” - a disease of immunocompromised laboratory mice.      
        </p>



       <br>


	   <p><img src="../assets/images/chaphama-header.jpg" alt="EPV-Dependo.54-Cavia and Myosin 9 homolog" /></p>


<blockquote>
		  Some of the species in which hamaparvoviruses and/or hamaparvovirus-derived EPVs have been identified.
		  Left to right: 'Icthamaparvovirus'-derived EPVs were identified in the tiger-tail seahorse;
		  <b>Porcine parvovirus 7</b> is an emerging virus of domestic pigs;
		  <b>Mouse kidney parvovirus</b> is associated with nephrotic disease in immunosuppressed laboratory mice;
		  We have identified EPVs derived from unclassified hamaparvovirus-like viruses in
		  a wide range of invertebrate species.
</blockquote>




       <br>


		<p>
		  We subsequently performed broader screening in animal genomes and identified 
		  EPV sequences derived from unclassified hamaparvovirus-like viruses in
		  arthropods and molluscs, as well as an <b>ichthamaparvovirus-derived EPV</b> identified in the 
		  genome of the tigertail seahorse.
		  <i>Ichthamaparvovirus</i> is the second genus defined in subfamily <i>Hamaparvovirinae</i>.
		  Officially it contains only a single species, Syngnathus scovelli chapparvovirus (ScChPV),
		  identified in the gulf pipefish (<i>Syngnathus scovelli</i>). However,
		  phylogenetic evidence supports the inclusion of 'Ichthyic parvovirus' in this genus. 

		</p>


	   <!-- ><p><img src="../assets/images/epv-ichthama.1-epv-ichthama.2.jpg" alt="Icthamaparvovirus EVE-related images" /></p>


<blockquote>
		  left to right: Pipefish, adder (Vipera berus), sea horse.       
          This sequence showed a close relationship to ScChPV - the prototype member of the 
          <i>Ichthamaparvovirus</i> genus.
</blockquote>

		
 -->

        </p>
           We recently identified icthamaparvovirus-derived EPVs in snakes,
           providing robust evidence that the host range of this viral genus extends to reptiles.
           Furthermore, orthologous copies of this EPV were identified in multiple snake species
           establishing that it integrated into the serpentine germline >50 million years ago.
           EPV-Icthama.2-Serpentes thus provides the most robust evidence yet that
           
           <a target="_blank" href="https://doi.org/10.3390/v11060525"><b>hamaparvoviruses are an ancient lineage</b></a>
           and have been associated with vertebrates throughout their evolution. 
        <p>

	
		<p>
           Related data items: 
		  <ol>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/tabular/eve/hamaparvovirinae/epv-ichthama-refseqs-side-data.tsv">Ichthama EPV reference sequence details</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/alignments/tips/ichthama/epv-ichthama.1-seasnakes.aln.fna">Ichthama ortholog sequences</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master/sources/genus/chaphama/fasta-refseqs-chaphama">Chaphamaparvovirus FASTA from WGS</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master/sources/genus/ichthama/fasta-digs-ichthama-epv">Ichthamaparvovirus FASTA from WGS</a></li>
		  </ol>

  		</p>



       <br>
	   <h3>
		 <a id="Amdo.1.Ellobius" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>EPVs: Amdo.1.Ellobius</strong>
	   </h3>
	   <hr>


		<p>
           Amdo.1.Ellobius is an amdoparvovirus-derived EPV, identified in the genome of
           the Transcaucasian mole vole (<i>Ellobius lutescens</i>) 
           - a species of cricetid rodent
           inhabiting semi-arid or grassland areas in Central Asia, and notable for
           its unusual karyotype: <b>only a single sex chromosome</b> is present - with the Y chromosome having been
           eliminated - and all individuals possess a diploid number of 17
           chromosomes.
           This interesting characteristic has motivated the sequencing of the <i>E.lutescens</i>
           genome, as well as that of a sister species - the northern mole vole
           (<i>E. talpinus</i>).
         </p>

          
 
		<p>
           We identified the corresponding empty genomic integration sites
           in <i>E. talpinus</i> indicating that both elements were incorporated into the
           <i>E. lutescens</i> germline within the last 10 million years.
           Intriguingly, in silico pedictions indicated that this replicase could be
           expressed as a fusion protein with a partial
           <a target="_blank" href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=MAFG"><b>MAFG</b></a> gene product.
        </p>

 

       <br>
         
 

        <p><img src="../assets/images/epv-amdo.1-caucasus.jpg" alt="Caucasus " /></p>



<blockquote>
		  Transcaucasus region - habitat of the Transcaucasian mole vole (<i>Ellobius lutescens</i>).
</blockquote>



       <br>

          
 
		<p>
           We identified four further EPV in mammal and reptile genomes that are intermediate
           between amdoparvoviruses and their sister genus (Protoparvovirus) in terms of
           their phylogenetic placement and genomic features. In particular, we identify
           a genome-length EPV in the genome of a pit viper (<i>Protobothrops mucrosquamatus</i>)
           that is intermediate between proto- and amdoparvoviruses. 
           Notably, it exhibits characteristically amdoparvovirus-like
           genome features including: (i) a putative middle ORF gene; (ii) a capsid gene
           that lacks a phospholipase A2 (PLA2) domain; (iii) a genome structure consistent
           with an amdoparvovirus-like mechanism of capsid gene expression.
         </p>

          
 
		<p>
           More recently, we identify orthologous copies of the Amdoparvovirus-like EPV
           in additional snake species, establishing it integrated into the genome
           >100 million years ago. Despite this, some copies encode a replicase gene that
           appears to have the potential to express intact protein.

   		</p>

		<p>
           Related data items: 
		  <ol>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/amdo/fasta-refseqs-amdo-epv/amdo.1-ellobius.fna">Ellobius sequence</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/amdo/fasta-refseqs-amdo-epv/ap.101-serpentes-UR.fna">Pit viper sequence</a></li>
			<li> <a target="_blank" href="../assets/images/epv-amdo-capsids.tif">Alignment and homology modelling of Pit Viper</a></li>
		  </ol>

  		</p>




       <br>
	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>
       <hr>

		<p>
		
		
		  Hildebrandt E, Penzes J, Gifford RJ, Agbandje-Mckenna M, and R Kotin
		  <b>(2020)</b>
		  <br>
		  Evolution of dependoparvoviruses across geological timescales – implications for design of AAV-based gene therapy vectors.
		  <i>Virus Evolution</i> 
		  <a target="_blank"  href="https://doi.org/10.1093/ve/veaa043" target="_blank">[view]</a> 
		  <br>
		  <br>

		  Pénzes JJ, de Souza WM, Agbandje-Mckenna M, and RJ Gifford
		  <b>(2019)</b>
		  <br>
		  An ancient lineage of highly divergent parvoviruses infects both vertebrate and invertebrate hosts.
		  <br>
		  <i>Viruses</i>
		  <a target="_blank"  href="https://doi.org/10.3390/v11060525" target="_blank">[view]</a> 
		  <br>
		  <br>
  
		  Callaway HM, Subramanian S, Urbina C, Barnard K, Dick R, Hafentein SL, Gifford RJ, and CR Parrish
		  <b>(2019)</b>
		  <br>
		  Examination and reconstruction of three ancient endogenous parvovirus capsid proteins in rodent genomes.
		  <br>
		  <i>Journal of Virology</i>
		  <a href="https://doi.org/10.1128/JVI.01542-18" target="_blank">[view]</a>
		  <br>
		  <br>

		  Valencia-Herrera I, Cena-Ahumada E, Faunes F, Ibarra-Karmy R, Gifford RJ*, and G Arriagada*                 
		  <b>(2019)</b>
		  *co-corresponding authors
		  <br>
		  Molecular properties and evolutionary origins of a parvovirus-derived myosin fusion gene in guinea pigs.
		  <br>
		  <i>Journal of Virology</i> <a href="https://doi.org/10.1128/JVI.00404-19" target="_blank">[view]</a>
		  <br>
		  <br>

		  Roediger B, Lee Q, Tikoo S, Cobbin JCA, Henderson JM, Jormakka M, O'Rourke MB, Padula MP, Pinello N,
		  Henry M, Wynne M, Santagostino SF, Brayton CF, Rasmussen L, Lisowski L, Tay SS, Harris DC, Bertram JF,
		  Dowling JP, Bertolino P, Lai JH, Wu W, Bachovchin WW, Wong JJ, Gorrell MD, Shaban B, Holmes EC, Jolly CJ,
		  Monette S, Weninger W.
		  <b>(2018)</b>
		  <br>
		  An Atypical Parvovirus Drives Chronic Tubulointerstitial Nephropathy and Kidney Fibrosis.
		  <i>Cell</i>. <a href="https://doi.org/10.1016/j.cell.2018.08.013" target="_blank">[view]</a>
		  <br>
		  <br>

		  Souza WM, Romeiro MF, Fumagalli MJ, Modha S, de Araujo J, Queiroz LH, Durigon EL, Figueiredo LT, Murcia PR, Gifford RJ.
		  <b>(2017)</b>
		  <br>
		  Chapparvoviruses occur in at least three vertebrate classes and have a broad biogeographic distribution.
		  <br>
		  <i>J Gen Virol.</i>
		  <a href="http://jgv.microbiologyresearch.org/content/journal/jgv/10.1099/jgv.0.000671" target="_blank">[view]</a>
		  <br>
		  <br>

		  Pénzes JJ, Marsile-Medun S, Agbandje-McKenna M, and RJ Gifford
		  <b>(2018)</b>
		  <br>
		  Endogenous amdoparvovirus-related elements reveal insights into the biology and evolution of vertebrate parvoviruses.
		  <br>
		  <i>Virus Evolution</i>
		  <a target="_blank"  href="https://academic.oup.com/ve/article/4/2/vey026/5179240" target="_blank">[view]</a> 
		  <br>
		  <br>

  		</p>



	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/">Parvovirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



