<!DOCTYPE html>
<html lang="en-us">

  <head>

    <meta charset="UTF-8">
    <title>Parvovirus-GLUE by giffordlabcvr: Paleoviruses</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Parvovirus-GLUE</h1>
      
      <h2 class="project-tagline">Resources for comparative genomic analysis of parvoviruses.</h2>

      <a href="../../index.html" class="btn">Home</a>
      <a href="./viruses.html" class="btn">Viruses</a>
      <a href="./highlights.html" class="btn">Highlights</a>
      <!-- <a href="./tutorial.html" class="btn">Tutorial</a>-->
      <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/tree/master" class="btn">GitHub</a>
    </section>

    <section class="main-content">

       
	   <h3>
		 <a id="Background" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Endogenous parvovirus (EPV) data in the Parvovirus-GLUE-EVE extension</strong>
	   </h3>
       <hr>

		<p>
          Whole genome sequencing has revealed that DNA sequences derived from
          parvoviruses are present within vertebrate genomes. These ‘<b>endogenous parvoviral elements</b>’
          (EPVs) are thought to have originated via ‘germline incorporation’ events in
          which parvovirus DNA sequences were integrated into chromosomal DNA of germline
          cells and subsequently inherited as novel host alleles. 
		</p>

		<br>
        <p><img src="../assets/images/parvo-paleovirus-header.png" alt="Parvovirus EVEs" /></p>
        <p><img src="../assets/images/parvo-paleovirus-midlevel.png" alt="Parvovirus EVEs" /></p>


<blockquote>
		<p>
		  Some of the species in which we identified <b>novel parvoviruses</b>
		  and <b>endogenous viral elements (EVEs)</b> derived from parvoviruses.
		  <b>Top row</b>, left to right: Masai giraffe (<i>Giraffa camelopardalis tippelskirchii)</i>),
		  Tasmanian devil (<i>Sarcophilus harrisii</i>),
		  elephants (<i>Elephantidae</i>),
		  chinchilla (<i>Chinchilla lanigera</i>).
		  <b>Bottom row</b>, left to right: Northern fur seals (<i>Callorhinus ursinus</i>), pit vipers (Crotalinae),
		  Leadbetter's possum (<i>Gymnobelideus leadbeateri</i>) , Transcaucasian mole vole  (<i>Ellobius lutescens</i>).
		</p>
</blockquote>
		 
 
		<br>

		<p>
          Analysis of EPVs has proven immensely informative with respect to the long-term
          evolutionary history of the Parvoviridae. EPV sequences are in some ways
          equivalent to parvovirus ‘fossils’ in that they provide a source of retrospective
          information about the distant ancestors of modern parvoviruses. 
		</p>

		<p>
          Currently, the distribution and diversity of parvovirus-related sequences
          in animal genomes remains incompletely characterized.
          Progress in characterising these elements has been hampered by the challenges
          encountered attempting to analyse their fragmentary and degenerated sequences. 
          <b>Parvovirus-GLUE</b> aims to address these issues.
          We have incorporated into this project a set of principles for
          organising the parvovirus 'fossil record', and a protocol 
          through which it can be accessed and collaboratively developed.
		</p>
 

		<p>
		  <b>Please note</b>: links to files on GitHub are mainly designed to indicate where
		  these files are located within the repository. 
		  To investigate files (e.g. tree files) in the appropriate software context we recommend
		  <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/zipball/master"><b>downloading the entire repository</b></a>
		  and browsing locally. 
		 
		</p>


        <br>

		<h3>
		  <a id="dataProvenance" class="anchor" href="#dataProvenance" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Where do the EPV data come from?</strong>
		</h3>
		<hr>
 

		<p>

		  EVE sequences were recovered from whole genome sequence (WGS) assemblies 
		  via <b>database-integrated genome screening (DIGS)</b> using the 
		   <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/"><b>DIGS tool</b></a>.
		</p>
	   


		<p>
		  All data pertaining to this screen are included in this repository.
		</p>

  
		<ul>

		<li>
		  The complete list of <b>vertebrate</b> genomes screened can be found
		  <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/digs/vertebrate-genomes-screened.csv"><b>here</b></a>.
		</li>

 
		<li>
		  The complete list of <b>invertebrate</b> genomes screened can be found
		  <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/digs/invertebrate-genomes-screened.csv"><b>here</b></a>.
		</li>
 
		<li>
		  The set of <b>parvovirus</b> polypeptide sequences used as probes can be found
		  <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/digs/digs-probes-parvo.faa"><b>here</b></a>.
		</li>
  
		<li>
		  The final set of <b>parvovirus</b> and <b>EPV</b> polypeptide sequences used as
		  references can be found

		  <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/digs/digs-reference-library/digs-reference-library-aa.fasta"><b>here</b></a>.
		</li>

		<li>
		  Input parameters for screening using the
		   <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/"><b>DIGS tool</b></a>
		  can be found
		  <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/digs/epv-screen-digs.ctl"><b>here</b></a>.
		</li>

		</ul>


		 <br>
		 <h3>
		   <a id="Nomenclature" class="anchor" href="#Nomenclature" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Standardised nomenclature for EPVs</strong>
		 </h3>
		 <hr>

		 <p>
		   We have applied a systematic approach to naming EPV, following a convention 
		   developed for <a target="_blank" href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1"><b>endogenous retroviruses (ERVs)</b></a>.
		   Each element was assigned a unique identifier (ID) constructed from a defined
		   set of components.
		 </p>

 
		 <p><img src="../assets/images/epv-nomenclature.png" alt="EPV Nomenclature" /></p>


		 <p>
		   The <b>first component</b> is the classifier ‘EPV’ (endogenous parvovirus element). 
		 </p>

		 <p>
		   The <b>second component</b> is a composite of two distinct subcomponents separated by a period:
		  
		   (i)  the name of EPV group;
		   (ii) a numeric ID that uniquely identifies the insertion.
		   The numeric ID is an integer identifies a unique insertion locus that arose as a
		   consequence of an initial germline infection. Thus, orthologous copies in different
		   species are given the same number. 
		 </p>
		
		 <p>
		   The <b>third component</b> of the ID defines the set of host species in which the ortholog occurs.
		 </p>



       <br>
	   <h3>
		 <a id="EPV reference sequences and data" class="anchor" href="#Ref_ecv_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>EPV reference sequences and data</strong>
	   </h3>
       <hr>


		<p>
           We reconstructed reference sequences for EPVs using alignments of EPV
           sequences derived from the same initial germline colonisation event - i.e.
           orthologous elements in distinct species, and paralogous 
           elements that have arisen via intragenomic duplication of EPV sequences.
  		</p>


		<p>
           Raw data in tabular format are can be found at the following links/directories: 
		  <ol>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/tabular/eve/parvovirinae/epv-amdo-refseqs-side-data.tsv">Amdoparvoviruses</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/tabular/eve/parvovirinae/epv-erythro-refseqs-side-data.tsv">Erythyroparvoviruses</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/tabular/eve/parvovirinae/epv-dependo-refseqs-side-data.tsv">Dependoparvoviruses</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/tabular/eve/parvovirinae/epv-proto-refseqs-side-data.tsv">Protoparvoviruses</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/tabular/eve/hamaparvovirinae/epv-ichthama-refseqs-side-data.tsv">Ichthamaparvoviruses</a></li>
		  </ol>

  		</p>

		<p>
           Nucleotide level data in FASTA format (individual files), can be found at the following links/directories:
 		  <ol>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/tree/master/sources/genus/amdo/fasta-refseqs-amdo-epv">Amdoparvoviruses</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/sources/genus/erythro/fasta-refseqs-erythro-epv">Erythyroparvoviruses</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/sources/genus/dependo/fasta-refseqs-dependo-epv">Dependoparvoviruses</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/sources/genus/proto/fasta-refseqs-proto-epv">Protoparvoviruses</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/sources/genus/ichthama/fasta-refseqs-ichthama-epv">Ichthamaparvoviruses</a></li>
		  </ol>
  		</p>
		



       <br>
	   <h3>
		 <a id="Multiple sequence alignments" class="anchor" href="#MultipleSequenceAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments - maps of homology between EPVs and viruses</strong>
	   </h3>
       <hr>

		<p>

          Multiple sequence alignment constructed in this study are linked together using
          GLUE's 
          ‘<a href="../assets/images/alignment tree concept small.png">alignment tree</a>’
          data structure. Alignments in the project include:

		  <ol>
			<li> A single ‘<a href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/alignments/root/parvo-ns-root+eves.aln"><strong>root</strong></a>’
			alignment constructed to represent proposed
			homologies between representative members of major parvovirus lineages
			(including extinct lineages represented only by EPVs).</li>
			<li> ‘<a href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/tree/master/alignments/internal/genus/"><strong>Genus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific parvovirus genera and EPV reference sequences.
			<li> ‘<a href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/tree/master/alignments/tips/"><strong>Tip</strong></a>’
			alignments in which all taxa are derived from a single EPV lineage.</li>
		  </ol>


 	   </p>


       <br>

	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees - reconstructed evolutionary relationships</strong>
	   </h3>
       <hr>


		<p>

          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the EPV-containing alignments included in our project,
          to reconstruct the evolutionary relationships between EPVs and related viruses.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
        
		  <ol>
			<li> Recursively populated <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/tree/master/trees/family-root"><strong>root</strong></a> phylogeny (Rep)</li>
			<li> <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/tree/master/trees/genus-level-with-eves/"><strong>Genus-level</strong></a> phylogenies</li>
			<li> <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/tree/master/trees/eve-orthologs/"><strong>EPV lineage-level</strong></a> phylogenies</li>
		  </ol>

 		</p>


		<br>
		<h3>
		  <a id="Raw EPV sequences and data" class="anchor" href="#Raw_digs_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Raw EPV sequences and data</strong>
		</h3>
		<hr>


		<p>
          These are the raw data generated by
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">database-integrated genome screening (DIGS)</a>.
          The tabular files contain information about the genomic location of each EVE. 
          EVEs were classified by comparison to a polypeptide sequence reference library
          designed to represent the known diversity of parvoviruses - this includes extinct 
          lineages represented only by endogenous viral elements (EVEs).
 		</p>

		<p>
          These data were obtained via
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">DIGS</a>
          performed in vertebrate genome assemblies downloaded from 
          <a href="https://www.ncbi.nlm.nih.gov/genome/browse ">NCBI genomes</a>
          (2020-07-15).
		</p>


		<p>
           Raw data in tabular format are can be found at the following links: 
		  <ol>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/tabular/eve/parvovirinae/epv-amdo-side-data.tsv">Amdoparvoviruses</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/tabular/eve/parvovirinae/epv-erythro-side-data.tsv">Erythyroparvoviruses</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/tabular/eve/parvovirinae/epv-dependo-side-data.tsv">Dependoparvoviruses</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/tabular/eve/parvovirinae/epv-proto-side-data.tsv">Protoparvoviruses</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/tabular/eve/hamaparvovirinae/epv-ichthama-side-data.tsv">Ichthamaparvoviruses</a></li>
		  </ol>

  		</p>

		<p>
           Nucleotide level data in FASTA format (individual files), can be found at the following links:
 		  <ol>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/tree/master/sources/genus/amdo/fasta-digs-amdo-epv">Amdoparvoviruses</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/sources/genus/erythro/fasta-digs-erythro-epv">Erythyroparvoviruses</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/sources/genus/dependo/fasta-digs-dependo-epv">Dependoparvoviruses</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/sources/genus/proto/fasta-digs-proto-epv">Protoparvoviruses</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/sources/genus/ichthama/fasta-digs-ichthama-epv">Ichthamaparvoviruses</a></li>
		  </ol>
  		</p>
		


       <br>
	   <h3>
		 <a id="schemaExtensions" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Paleovirus-specific schema extensions</strong>
	   </h3>
       <hr>


		 <p>
		   The paleovirus component of Parvovirus-GLUE extends GLUE's
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/coreSchema"><b>core schema</b></a>
		   to allow the capture of EVE-specific data.
		   These schema extensions are defined in 
           <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/blob/master/glue/build/core/parvoSchemaExtensions.glue"><b>this file</b></a>
		   and comprise two additional table: 'locus_data' and 'refcon_data'.
		   Both tables are linked to the main 'sequence' table via the 'sequenceID' field.
  		</p>


 		<p>
		   The '<b>locus_data</b>' table contains EVE locus information: e.g. species, assembly, scaffold, location coordinates.
		</p>

 		<p>
		   The '<b>refcon_data</b>' table contains summary information for individual
		   EVE insertions. It refers to the reference sequences constructed to represent
		   each insertion, which reflect our best efforts to reconstruct progenitor virus
		   sequences as they might have looked when they initially integrated into 
		   the germline of ancestral species. 
		</p>


		<br>
		<h3>
		  <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
		</h3>
		<hr>

		 <p>
  
		  Campbell M, Loncar S, Gifford RJ, Kotin R, and RJ Gifford
		  <b>(2022)</b>
		  <br>
		  Comparative analysis reveals the long-term co-evolutionary history of parvoviruses and vertebrates.
		  <br>
		  <i>PLoS Biology</i> 
		  <a href="https://doi.org/10.1371/journal.pbio.3001867">[view]</a> 
		  <br>
		  <br>
		 
		 Hildebrandt E, Penzes J, Gifford RJ, Agbandje-Mckenna M, and R Kotin
		 <b>(2020)</b>
		 <br>
		 Evolution of dependoparvoviruses across geological timescales – implications for design of AAV-based gene therapy vectors.
		 <i>Virus Evolution</i> 
		 <a target="_blank"  href="https://academic.oup.com/ve/article/doi/10.1093/ve/veaa043/5842153">[view]</a> 
		 <br>
		 <br>

		 Pénzes JJ, de Souza WM, Agbandje-Mckenna M, and RJ Gifford
		 <b>(2019)</b>
		 <br>
		 An ancient lineage of highly divergent parvoviruses infects both vertebrate and invertebrate hosts.
		 <br>
		 <i>Viruses</i>
		 <a target="_blank"  href="https://www.mdpi.com/1999-4915/11/6/525" target="_blank">[view]</a> 
		 <br>
		 <br>


		 Callaway HM, Subramanian S, Urbina C, Barnard K, Dick R, Hafentein SL, Gifford RJ, and CR Parrish
		 <b>(2019)</b>
		 <br>
		 Examination and reconstruction of three ancient endogenous parvovirus capsid proteins in rodent genomes.
		 <br>
		 <i>Journal of Virology</i>
		 <a href="https://jvi.asm.org/content/93/6/e01542-18.long" target="_blank">[view]</a>
		 <br>
		 <br>


		 Kobayashi Y, Shimazu T, Murata K, Itou T, Suzuki Y.
		 <b>(2019)</b>
		 <br>
		 An endogenous adeno-associated virus element in elephants. 
		 <br>
		 <i> Virus Res.</i> Mar;262:10-14
		 <a target="_blank"  href="https://pubmed.ncbi.nlm.nih.gov/29702129/" target="_blank">[view]</a> 
		 <br>
		 <br>


		 Valencia-Herrera I, Cena-Ahumada E, Faunes F, Ibarra-Karmy R, Gifford RJ*, and G Arriagada*                 
		 <b>(2019)</b>
		 *co-corresponding authors
		 <br>
		 Molecular properties and evolutionary origins of a parvovirus-derived myosin fusion gene in guinea pigs.
		 <br>
		 <i>Journal of Virology</i> <a href="https://jvi.asm.org/content/early/2019/06/13/JVI.00404-19.long" target="_blank">[view]</a>
		 <br>
		 <br>

		 Pénzes JJ, Marsile-Medun S, Agbandje-McKenna M, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 Endogenous amdoparvovirus-related elements reveal insights into the biology and evolution of vertebrate parvoviruses.
		 <br>
		 <i>Virus Evolution</i>
		 <a target="_blank"  href="https://academic.oup.com/ve/article/4/2/vey026/5179240" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 GLUE: A flexible software system for virus sequence data.
		 <br>
		 <i>BMC Bioinformatics</i>
		 <a target="_blank"  href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Zhu H, Dennis T, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		 <a target="_blank"  href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		 <br>
		 <br>

		 Gifford RJ, Blomberg B, Coffin JM, Fan H, Heidmann T, Mayer J, Stoye J, Tristem M, and WE Johnson
		 <b>(2018)</b>
		 <br>
		 Nomenclature for endogenous retrovirus (ERV) loci.
		 <br>
		 <i>Retrovirology</i>
		 <a target="_blank"  href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Gloria Arriagada and RJ Gifford
		 <b>(2014)</b>
		 <br>
		 Parvovirus-derived endogenous viral elements in two South American rodent genomes.
		 <br>
		 <i>J. Virol.</i>
		 <a target="_blank"  href="http://jvi.asm.org/content/88/20/12158.long" target="_blank">[view]</a>
		 <br>
		 <br>

		 Katzourakis A. and RJ. Gifford
		 <b>(2010)</b>
		 <br>
		 Endogenous viral elements in animal genomes.
		 <br>
		 <i>PLoS Genetics</i>
		 <a target="_blank"  href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1001191" target="_blank">[view]</a>

		</p>




	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Parvovirus-GLUE-EVE/">Parvovirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



