<!DOCTYPE html>
<html lang="en-us">

  <head>

    <meta charset="UTF-8">
    <title>Parvovirus-GLUE by giffordlabcvr: Viruses</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Parvovirus-GLUE</h1>
 
      
      <h2 class="project-tagline">Resources for comparative genomic analysis of parvoviruses.</h2>

      <a href="../../index.html" class="btn">Home</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="./highlights.html" class="btn">Highlights</a>
      <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master" class="btn">View on GitHub</a>
      
    </section>

    <section class="main-content">




	   <h3>
		 <a id="VirusProject" class="anchor" href="#GlueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Virus data included in Parvovirus-GLUE</strong>
	   </h3>
	   <hr>
	   

	   <p>
		 This page provides background information on the <b>virus-associated</b>
		 data items included in the project  - information about endogenous parvoviral elements (EPVs)
		 can be found <a href="./paleoviruses.html""><b>here</b></a>.
	   </p>



	   <h3>
		 <a id="genomeFeatures" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Parvovirus genome features</strong>
	   </h3>
	   <hr>


	   <p>

         Parvoviruses have linear, single-stranded DNA genomes ~5 kilobases (kb)
         in length. They are typically very compact and generally exhibit the same basic
         genetic organisation comprising two major gene cassettes, one (<b>Rep/NS</b>) that encodes the
         non-structural proteins, and another (<b>Cap/VP</b>) that encodes the structural
         coat proteins of the virion.
	   </p>

	   <br>


        <p><img src="../assets/images/cpv-genome.png" alt="CPV genome" /></p>
 
<blockquote>
		<p>
		   A schematic representation of the canine parvovirus (CPV) genome. NS=non-structural;
		   VP=capsid; PLA2=phospholipase A2; ITR=inverted terminal repeat; Kb=kilobases
		</p>
         
</blockquote>

	   <br>

	   <p>
         Some species and genera encode additional polypeptide gene products adjacent to
         these genes or overlapping them in alternative reading frames. 
		</p>
         


		<p>
         The genome is flanked at the 3' and 5' ends by palindromic <b>inverted terminal repeat (ITR)</b>
         sequences that are the only <i>cis</i> elements required for replication. 

	   </p>

	   <p>
	   
		 <b>Parvovirus-GLUE</b> defines a 
		 <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/glue/build/core/parvoFeatures.glue"><b>standard set of genome features</b></a>
		 for parvoviruses and
		 <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/glue/build/core/parvoMasterReferences.glue">records the locations</b></a>
		 of these genome features on master reference sequences (see below).
	   
	   </p>


	   <br>
	   <h3>
		 <a id="sequenceData" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Sequences and sequence-associated data</strong>
	   </h3>
       <hr>


	   <p>
		 The sequence data in this project are organised into multiple distinct sources.
		 Each source contains data in either GenBank XML or plain FASTA format.
		 The type of data is indicated by the name of the source (all GenBank XML sources 
		 contain 'ncbi' in the name).
	   </p>
	   

	   <p>
         The following NCBI-derived sources are included in the project:


		 <ul>
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/core/ncbi-refseqs"><b>ncbi-refseqs</b></a>: Core project master reference sequences (one per parvovirus genus)</li>
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/amdo/ncbi-refseqs-amdo"><b>ncbi-refseqs-amdo</b></a>: Amdoparvovirus </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/proto/ncbi-refseqs-proto"><b>ncbi-refseqs-proto</b></a>: Protoparvovirus </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/boca/ncbi-refseqs-boca"><b>ncbi-refseqs-boca</b></a>: Bocaparvovirus </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/ave/ncbi-refseqs-ave"><b>ncbi-refseqs-ave</b></a>: Aveparvovirus </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/tetra/ncbi-refseqs-tetra"><b>ncbi-refseqs-tetra</b></a>: Tetraparvovirus</li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/copi/ncbi-refseqs-copi"><b>ncbi-refseqs-copi</b></a>: Copiparvovirus </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/erythro/ncbi-refseqs-erythro"><b>ncbi-refseqs-erythro</b></a>: Erythroparvovirus </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/dependo/ncbi-refseqs-dependo"><b>ncbi-refseqs-dependo</b></a>: Dependoparvovirus </li> 		   	   
		 </ul>
		 
		 
	   </p>




	   <p>
		 GenBank XML files are imported into this project directly from <a href="https://www.ncbi.nlm.nih.gov/genbank/">NCBI GenBank</a>
		 using <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/modules/core/parvoGenbankXmlPopulator.xml">an appropriately configured version</a>		 
		 of GLUE's 'GenBankPopulator' module and are uniquely identified within this project by their
		 <a target="_blank" href="https://www.ncbi.nlm.nih.gov/Sequin/acc.html">GenBank accession numbers</a>. 

	   </p>

	   <p>
         Sequences included in this project are linked to auxiliary data in tabular format, as follows:

	   </p>


       <br>

	   
	   <table>
	
		   <thead>
			   <tr>
				   <th>Parameter</th>
				   <th>Type</th>
				   <th>Definition</th>
			   </tr>
		   </thead>
	
		   <tbody>

			   <tr>
				   <td><i>full_name</i></td>
				   <td>VARCHAR</td>
				   <td>Full name of the virus this sequence is derived from</td>
			   </tr>

			   <tr>
				   <td><i>name</i></td>
				   <td>VARCHAR</td>
				   <td>Abbreviated name of the virus this sequence is derived from</td>
			   </tr>

			   <tr>
				   <td><i>subfamily</i></td>
				   <td>VARCHAR</td>
				   <td>Taxonomy - virus subfamily</td>
			   </tr>

			   <tr>
				   <td><i>supergenus</i></td>
				   <td>VARCHAR</td>
				   <td>Taxonomy - virus supergenus (proposed)</td>
			   </tr>

			   <tr>
				   <td><i>genus</i></td>
				   <td>VARCHAR</td>
				   <td>Taxonomy - virus genus</td>
			   </tr>

			   <tr>
				   <td><i>clade</i></td>
				   <td>VARCHAR</td>
				   <td>Taxonomy - virus clade</td>
			   </tr>

			   <tr>
				   <td><i>isolate_name</i></td>
				   <td>VARCHAR</td>
				   <td>Name of the isolate this sequence is derived from</td>
			   </tr>

			   <tr>
				   <td><i>isolation_host</i></td>
				   <td>VARCHAR</td>
				   <td>Species (latin binomial) virus was isolated from</td>
			   </tr>

			   <tr>
				   <td><i>length</i></td>
				   <td>INTEGER</td>
				   <td>Length of the sequence</td>
			   </tr>

			   <tr>
				   <td><i>pubmed_id</i></td>
				   <td>INTEGER</td>
				   <td>PubMed ID of manuscript associated with sequence</td>
			   </tr>

			   <tr>
				   <td><i>gb_create_date</i></td>
				   <td>GenBank</td>
				   <td>GenBank creation date of the sequence</td>
			   </tr>

			   <tr>
				   <td><i>gb_update_date</i></td>
				   <td>VARCHAR</td>
				   <td>Date of most recent GenBank update</td>
			   </tr>

			   <tr>
				   <td><i>country</i></td>
				   <td>VARCHAR</td>
				   <td>Country where virus was isolated</td>
			   </tr>

			   <tr>
				   <td><i>place_sampled</i></td>
				   <td>VARCHAR</td>
				   <td>Location of sampling (state, region, or city)</td>
			   </tr>

			   <tr>
				   <td><i>collection_year</i></td>
				   <td>INTEGER</td>
				   <td>Year virus was isolated</td>
			   </tr>

			   <tr>
				   <td><i>collection_month</i></td>
				   <td>VARCHAR</td>
				   <td>Month virus was isolated</td>
			   </tr>


			   <tr>
				   <td><i>collection_month_day</i></td>
				   <td>VARCHAR</td>
				   <td>Day of month virus was isolated</td>
			   </tr>


		   </tbody>
		
	   </table>


 
	   <br>
	   <h3>
		 <a id="virusReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Parvovirus reference sequences</strong>
	   </h3>
	   <hr>
	   
		<p>

 		  For all offially recognised parvoviral genera, we defined a 'master' reference sequence, as follows:


		  <h4>
			<a id="Parvovirinae" class="anchor" href="#Epiparvovirinae" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Parvovirinae</strong>
		  </h4>


		  <ul>
		    <li><b><i>Protoparvovirus</i></b>: Carnivore protoparvovirus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001539"><b>NC_001539</b></a>)</li>
		    <li><b><i>Aveparvovirus</i></b>: Chicken parvovirus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_024452"><b>NC_024452</b></a>)</li>
		    <li><b><i>Amdoparvovirus</i></b>: Carnivore amdoparvovirus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001662"><b>NC_001662</b></a>)</li>
		    <li><b><i>Erythroparvovirus</i></b>: Human parvovirus B19 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_000883"><b>NC_000883</b></a>)</li>
		    <li><b><i>Dependoparvovirus</i></b>: Adeno-associated virus 2 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001401"><b>NC_001401</b></a>)</li>
		    <li><b><i>Copiparvovirus</i></b>: Bovine parvovirus 2 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_006259"><b>NC_006259</b></a>) Bovine parvovirus 2</li>
		    <li><b><i>Bocaparvovirus</i></b>: Bovine parvovirus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001540"><b>NC_001540</b></a>)</li>
		    <li><b><i>Tetraparvovirus</i></b>: Human parvovirus 4 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_007018"><b>NC_007018</b></a>)</li>
		    <li><b><i>Artiparvovirus</i></b>: Artibeus jamaicensis parvovirus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_016752"><b>NC_016752</b></a>)</li>
		  </ul>

		  <h4>
			<a id="Hamaparvovirinae" class="anchor" href="#Epiparvovirinae" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Hamaparvovirinae</strong>
		  </h4>

		  <ul>
		    <li><b><i>Chaphamaparvovirus</i></b>: Porcine parvovirus 7 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_040562"><b>NC_040562</b>)</a></li>
		    <li><b><i>Ichthamaparvovirus</i></b>: Syngnathus scovelli chapparvovirus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/MN049932"><b>MN049932</b>)</a></li>
		    <li><b><i>Brevidensoparvovirus</i></b>: Aedes albopictus densovirus 2 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_004285"><b>NC_004285</b>)</a></li>
		    <li><b><i>Penstyldensoparvovirus</i></b>: Infectious hypodermal and hematopoietic necrosis virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_002190"><b>NC_002190</b>)</a></li>
		  </ul>



		  <h4>
			<a id="Densoparvovirinae" class="anchor" href="#Epiparvovirinae" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Densoparvovirinae</strong>
		  </h4>

		  <ul>
		    <li><b><i>Ambidensovirus</i></b>: Junonia coenia densovirus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_004284"><b>NC_004284</b>)</a></li>
		    <li><b><i>Aquambidensovirus</i></b>:  Asteroid aquambidensovirus 1  (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_038532"><b>NC_038532</b>)</a></li>
		    <li><b><i>Blattambidensovirus</i></b>:  Blattodean blattambidensovirus 1  (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_005041"><b>NC_005041</b>)</a></li>
		    <li><b><i>Diciambidensovirus</i></b>:  Hemipteran diciambidensovirus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_030296"><b>NC_030296</b>)</a></li>
		    <li><b><i>Hemiambidensovirus</i></b>:  Dysaphis plantaginea densovirus   (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_022564"><b>NC_034532</b>)</a></li>
		    <li><b><i>Iteradensoparvovirus</i></b>: Lepidopteran iteradensovirus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_003346"><b>NC_003346</b>)</a></li>
		    <li><b><i>Miniambidensovirus</i></b>:  Orthopteran miniambidensovirus 1  (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_022564"><b>NC_022564</b>)</a></li>
		    <li><b><i>Muscambidensovirus</i></b>:  Haematobia irritans densovirus  (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/MK643151"><b>MK643151</b>)</a></li>
		    <li><b><i>Pefuambidensovirus</i></b>:  Blattodean pefuambidensovirus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_000936"><b>NC_000936</b>)</a></li>
		    <li><b><i>Protoambidensovirus</i></b>:  Dipteran protoambidensovirus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/MK722617"><b>MK722617</b>)</a></li>
		    <li><b><i>Scindoambidensovirus</i></b>: Hemipteran scindoambidensovirus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_004289"><b>NC_004289</b>)</a></li>
		  </ul>

		</p>


         
		<p>

          We explicitly defined the locations of genome features
          on master reference sequences
          (<b>see <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/glue/build/core/parvoMasterReferences.glue">here</b></a>).
		  
 		</p>
        


	   <br>
	   <h3>
		 <a id="Multiple sequence alignments" class="anchor" href="#MultipleSequenceAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments (MSAs)</strong>
	   </h3>
	   <hr>

		<p>
          <b>Multiple sequence alignments (MSAs)</b> are the basic currency of comparative genomic analysis.
          MSAs constructed in this study are linked together using
          GLUE's <b>constrained MSA tree</b> data structure.
  		</p>

         
 		<p>
          A 'constrained MSA' is an alignment in which the coordinate space is defined by 
          a selected reference sequence. Where alignment members contain insertions relative
          to the reference sequence, the inserted sequences are recorded and stored
          (i.e. sequence data is never deleted).
  		</p>

         
 		<p>
           
          GLUE projects have the option of using a data structure called an <b>alignment tree</b>
          to link constrained MSAs representing different taxonomic levels,
          and we've used this approach in Parvovirus-GLUE.
 		</p>



 	    <br>



        
         <p><img src="../assets/images/Parvovirus-GLUE-header.jpg" alt="Parvovirus phylogeny" /></p>
 

<blockquote>

		 <p>
		   The
		   <a target="_blank" href="https://evolution-outreach.biomedcentral.com/articles/10.1007/s12052-010-0273-6"><b>phylogenetic tree</b></a>
		   shown above, taken from a report by
		   <a target="_blank" href="https://dx.doi.org/10.1007/s00705-020-04632-4"><b>Pénzes <i>et al.</i> (2020)</b></a>,	    
		   shows the evolutionary relationships between currently recognised genera in  
		   the <b>family <i>Parvoviridae</i></b>. 
		 </p>
         
</blockquote>






 	    <br>

        <p><img src="../assets/images/parvo-msa-tree.jpg" alt="Alignment tree concept" /></p>
 
 		

<blockquote>
		<p>
		  The schematic figure above shows the '<b>alignment tree</b>' data structure 
		  currently implemented in <b>Parvovirus-GLUE</b>.
          For the highest taxonomic levels (i.e. at the root) we aligned only the most
          conserved regions of the genome, whereas for the lower
          taxonomic levels (i.e. within and below genus level) we aligned complete coding
          sequences.

          We used an alignment tree data structure to link these alignments,
          via a set of common reference sequences.
          The root alignment contains reference sequences for major clades,
          whereas all children of the
          root inherit at least one reference from their immediate parent.
          Thus, all alignments are linked to one another via our chosen set of
          master reference sequences.
		</p>
         
</blockquote>


 	    <br>
   
		<p>
             
          
          Alignments imported into in the project include:
         
		  
		  <ol>
			<li> A ‘<a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/alignments/root/parvo-ns-root.aln.fna"><strong>root</strong></a>’
			alignment constructed to represent homology between the two largest subgroupings in the <i>Parvoviridae</i>.
			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master/alignments/internal/subfamily"><strong>subfamily</strong></a>’
			alignments constructed to represent proposed
			homologies between representative members of <i>Parvoviridae</i> subfamilies</li>

			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master/alignments/internal/cross-genus"><strong>cross-genus</strong></a>’
			alignments constructed to represent proposed
			homologies between representative members of 'minor' <i>Parvoviridae</i> lineages</li>

			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master/alignments/internal/genus/"><strong>genus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific parvovirus genera.</li>
		  </ol>
         
 		</p>



	   <br>
	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees</strong>
	   </h3>
	   <hr>


		<p>

          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
        
		  <ol>
			<li> <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master/trees/family-root"><strong>Family-level (root)</strong></a> phylogeny (Rep)</li>
			<li> <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master/trees/genus-level"><strong>Genus-level</strong></a> phylogenies</li>
		  </ol>

 		</p>






	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/">Parvovirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



