<!DOCTYPE html>
<html lang="en-us">

  <head>

    <meta charset="UTF-8">
    <title>Parvovirus-GLUE by giffordlabcvr: Viruses</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Parvovirus-GLUE</h1>
 
      
      <h2 class="project-tagline">Resources for comparative genomic analysis of parvoviruses.</h2>

      <a href="../../index.html" class="btn">Home</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="./highlights.html" class="btn">Highlights</a>
      <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master" class="btn">View on GitHub</a>
      
    </section>

    <section class="main-content">




	   <h3>
		 <a id="VirusProject" class="anchor" href="#GlueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Comparative genomic analysis of parvoviruses using Parvovirus-GLUE</strong>
	   </h3>
	   <hr>
	   

        
       <p><img src="../assets/images/parvovirus-header.jpg" alt="Parvoviruses"/></p>
         
	   

<blockquote>
		 <p>
		   Comparative genomic analysis of parvoviruses can help reveal the biological basis
		   of parvovirus replication.
		   The insights obtained via comparative analysis of parvovirus genomes can directly
		   inform efforts to limit parvovirus-associated disease and develop parvovirus-based 
		   therapeutics. 
		 </p>
         
</blockquote>



	   <p>
	     Since the emergence of the SARS-COV2 pandemic, many have become familiar with 
	     the use of virus genome data to track the spread and evolution of pathogenic viruses
	     - e.g. via tools such as <a target="_blank" href="https://nextstrain.org/ncov/gisaid/global""><b>NextStrain</b></a>. 
		 However, comparative genomic approaches can also be used to reveal the structural and functional
		 basis of virus adaptations. 
	   </p>


	   <p>
		 <b>Parvovirus-GLUE</b> provides a highly flexible, extensible platform for
		 implementing computational genomic analysis of parvoviruses
		 in an <b>efficient, standardised and reproducible</b> way.
		 It is equally suited for research purposes as it is for more operational procedures
		 (e.g. producing standardised, genome data-based reports in a public or animal health setting)
	   </p>



	   <p>
		 This page provides background information on the <b>virus-associated</b>
		 data items included in the project  - information about endogenous parvoviral elements (EPVs)
		 can be found <a href="./paleoviruses.html""><b>here</b></a>.
	   </p>




	   <h3>
		 <a id="sequenceData" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Sequences and associated data</strong>
	   </h3>
       <hr>


	   <p>
		 The sequence data in this project are organised into multiple distinct sources.
		 Each source contains data in either GenBank XML or plain FASTA format.
		 The type of data is indicated by the name of the source (all GenBank XML sources 
		 contain 'ncbi' in the name).
	   </p>
	   

	   <p>
		 GenBank XML files are imported into this project directly from <a href="https://www.ncbi.nlm.nih.gov/genbank/">NCBI GenBank</a> using
		 a GLUE module and are uniquely identified within this project by their <a target="_blank" href="https://www.ncbi.nlm.nih.gov/Sequin/acc.html">GenBank accession numbers</a>. 

		 Where possible, we prefer to use sequences obtained via GenBank since it
		 represents the principle source of published nucleotide sequence data.
		 However, FASTA sources can also be used in GLUE,
		 making it straightforward to expand the curated, GenBank-derived set with 
		 unpublished sequences. 
	   </p>



	   <p>
         This project includes the following NCBI-derived sources:


		 <ul>
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/core/ncbi-refseqs"><b>ncbi-refseqs</b></a>: Core project master reference sequences (one per parvovirus genus)</li>
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/amdo/ncbi-refseqs-amdo"><b>ncbi-refseqs-amdo</b></a>: Amdoparvovirus </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/proto/ncbi-refseqs-proto"><b>ncbi-refseqs-proto</b></a>: Protoparvovirus </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/boca/ncbi-refseqs-boca"><b>ncbi-refseqs-boca</b></a>: Bocaparvovirus </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/ave/ncbi-refseqs-ave"><b>ncbi-refseqs-ave</b></a>: Aveparvovirus </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/tetra/ncbi-refseqs-tetra"><b>ncbi-refseqs-tetra</b></a>: Tetraparvovirus</li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/copi/ncbi-refseqs-copi"><b>ncbi-refseqs-copi</b></a>: Copiparvovirus </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/erythro/ncbi-refseqs-erythro"><b>ncbi-refseqs-erythro</b></a>: Erythroparvovirus </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/sources/genus/dependo/ncbi-refseqs-dependo"><b>ncbi-refseqs-dependo</b></a>: Dependoparvovirus </li> 		   	   
		 </ul>
		 
		 
	   </p>




	   <p>
         Sequences included in this project are linked to auxiliary data in tabular format:


		 <ul>
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/tabular/core/parvo-ncbi-refseqs-side-data.tsv"><b>Core project master references</b></a></li>
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/tabular/genus/amdo-refseqs-side-data.tsv"><b>Amdoparvovirus</b></a> </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/tabular/genus/proto-refseqs-side-data.tsv"><b>Protoparvovirus</b></a> </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/tabular/genus/boca-refseqs-side-data.tsv"><b>Bocaparvovirus</b></a> </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/tabular/genus/ave-refseqs-side-data.tsv"><b>Aveparvovirus</b></a> </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/tabular/genus/tetra-refseqs-side-data.tsv"><b>Tetraparvovirus</b></a> </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/tabular/genus/copi-refseqs-side-data.tsv"><b>Copiparvovirus</b></a> </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/tabular/genus/erythro-refseqs-side-data.tsv"><b>Erythroparvovirus</b></a> </li> 
		   <li><a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/tabular/genus/dependo-refseqs-side-data.tsv"><b>Dependoparvovirus</b></a> </li> 
		   	   
		 </ul>


	   </p>


	   <br>
	   <h3>
		 <a id="genomeFeatures" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Lentivirus genome features</strong>
	   </h3>
	   <hr>

		<p>
          Lentiviruses have enveloped, spherical virions and a small, circular DNA
          genome ~3 kilobases (Kb) in length. The genome is characterised by a highly
          streamlined organization incorporating extensive gene overlap - the open reading
          frame (ORF) encoding the viral polymerase (P) protein occupies most of the genome
          and typically overlaps at least one of the ORFs encoding the core (C), and
          surface (S) proteins. 
 		</p>
		
		<p>
         We defined a
         <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/glue/core/lentiFeatures.glue"><b>standard set of genome features</b></a>
		 for lentiviruses and the locations of these genome features
		 on master reference sequences
		 (<b>see <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/glue/core/lentiMasterReferences.glue">here</b></a>).
		</p>


	   <br>


	   <h3>
		 <a id="virusReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Parvovirus reference sequences</strong>
	   </h3>
	   <hr>



	   
		<p>

 		  For all offially recognised parvoviral genera, we defined a 'master' reference sequence, as follows:


	   <h4>
		 <a id="Parvovirinae" class="anchor" href="#Epiparvovirinae" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Parvovirinae</strong>
	   </h4>


		  <ul>
		    <li><b><i>Protoparvovirus</i></b>: Carnivore protoparvovirus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001539"><b>NC_001539</b></a>)</li>
		    <li><b><i>Aveparvovirus</i></b>: Chicken parvovirus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_024452"><b>NC_024452</b></a>)</li>
		    <li><b><i>Amdoparvovirus</i></b>: Carnivore amdoparvovirus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001662"><b>NC_001662</b></a>)</li>
		    <li><b><i>Erythroparvovirus</i></b>: Human parvovirus B19 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_000883"><b>NC_000883</b></a>)</li>
		    <li><b><i>Dependoparvovirus</i></b>: Adeno-associated virus 2 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001401"><b>NC_001401</b></a>)</li>
		    <li><b><i>Copiparvovirus</i></b>: Bovine parvovirus 2 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_006259"><b>NC_006259</b></a>) Bovine parvovirus 2</li>
		    <li><b><i>Bocaparvovirus</i></b>: Bovine parvovirus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001540"><b>NC_001540</b></a>)</li>
		    <li><b><i>Tetraparvovirus</i></b>: Human parvovirus 4 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_007018"><b>NC_007018</b></a>)</li>
		    <li><b><i>Artiparvovirus</i></b>: Artibeus jamaicensis parvovirus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_016752"><b>NC_016752</b></a>)</li>
		  </ul>

		</p>




	   <h4>
		 <a id="Hamaparvovirinae" class="anchor" href="#Epiparvovirinae" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Hamaparvovirinae</strong>
	   </h4>

		  <ul>
		    <li><b><i>Chaphamaparvovirus</i></b>: Porcine parvovirus 7 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_040562"><b>NC_040562</b>)</a></li>
		    <li><b><i>Ichthamaparvovirus</i></b>: Syngnathus scovelli chapparvovirus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/MN049932"><b>MN049932</b>)</a></li>
		    <li><b><i>Brevidensoparvovirus</i></b>: Aedes albopictus densovirus 2 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_004285"><b>NC_004285</b>)</a></li>
		    <li><b><i>Penstyldensoparvovirus</i></b>: Infectious hypodermal and hematopoietic necrosis virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_002190"><b>NC_002190</b>)</a></li>
		  </ul>



	   <h4>
		 <a id="Densoparvovirinae" class="anchor" href="#Epiparvovirinae" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Densoparvovirinae</strong>
	   </h4>

		  <ul>
		    <li><b><i>Ambidensovirus</i></b>: Junonia coenia densovirus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_004284"><b>NC_004284</b>)</a></li>
		    <li><b><i>Aquambidensovirus</i></b>:  Asteroid aquambidensovirus 1  (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_038532"><b>NC_038532</b>)</a></li>
		    <li><b><i>Blattambidensovirus</i></b>:  Blattodean blattambidensovirus 1  (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_005041"><b>NC_005041</b>)</a></li>
		    <li><b><i>Diciambidensovirus</i></b>:  Hemipteran diciambidensovirus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_030296"><b>NC_030296</b>)</a></li>
		    <li><b><i>Hemiambidensovirus</i></b>:  Dysaphis plantaginea densovirus   (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_022564"><b>NC_034532</b>)</a></li>
		    <li><b><i>Iteradensoparvovirus</i></b>: Lepidopteran iteradensovirus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_003346"><b>NC_003346</b>)</a></li>
		    <li><b><i>Miniambidensovirus</i></b>:  Orthopteran miniambidensovirus 1  (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_022564"><b>NC_022564</b>)</a></li>
		    <li><b><i>Muscambidensovirus</i></b>:  Haematobia irritans densovirus  (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/MK643151"><b>MK643151</b>)</a></li>
		    <li><b><i>Pefuambidensovirus</i></b>:  Blattodean pefuambidensovirus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_000936"><b>NC_000936</b>)</a></li>
		    <li><b><i>Protoambidensovirus</i></b>:  Dipteran protoambidensovirus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/MK722617"><b>MK722617</b>)</a></li>
		    <li><b><i>Scindoambidensovirus</i></b>: Hemipteran scindoambidensovirus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_004289"><b>NC_004289</b>)</a></li>


		  </ul>

         
		<p>

          We explicitly defined the locations of genome features
          on master reference sequences
          (<b>see <a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/glue/build/core/parvoMasterReferences.glue">here</b></a>).
		  
 		</p>
        


	   <br>
	   <h3>
		 <a id="Multiple sequence alignments" class="anchor" href="#MultipleSequenceAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments (MSAs)</strong>
	   </h3>
	   <hr>

		<p>
          Multiple sequence alignments (MSAs) are the basic currency of comparative genomic analysis.
          MSAs constructed in this study are linked together using
          GLUE's <b>constrained MSA tree</b> data structure.
  		</p>

         
 		<p>
          A 'constrained MSA' is an alignment in which the coordinate space is defined by 
          a selected reference sequence. Where alignment members contain insertions relative
          to the reference sequence, the inserted sequences are recorded and stored
          (i.e. sequence data is never deleted).
  		</p>

         
 		<p>
           
          GLUE projects have the option of using a data structure called an <b>alignment tree</b>
          to link constrained MSAs representing different taxonomic levels,
          and we've used this approach in Parvovirus-GLUE.
 		</p>


 	    <br>

        <p><img src="../assets/images/parvo-msa-tree.jpg" alt="Alignment tree concept" /></p>
 
 		

<blockquote>
		<p>
		  The schematic figure above shows the '<b>alignment tree</b>' data structure 
		  currently implemented in <b>Parvovirus-GLUE</b>.
          For the highest taxonomic levels (i.e. at the root) we aligned only the most
          conserved regions of the genome, whereas for the lower
          taxonomic levels (i.e. within and below genus level) we aligned complete coding
          sequences.

          We used an alignment tree data structure to link these alignments,
          via a set of common reference sequences.
          The root alignment contains reference sequences for major clades,
          whereas all children of the
          root inherit at least one reference from their immediate parent.
          Thus, all alignments are linked to one another via our chosen set of
          master reference sequences.
		</p>
         
</blockquote>


 	    <br>
   
		<p>
             
          
          Alignments imported into in the project include:
         
		  
		  <ol>
			<li> A ‘<a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/alignments/root/parvo-ns-root.aln.fna"><strong>root</strong></a>’
			alignment constructed to represent homology between the two largest subgroupings in the <i>Parvoviridae</i>.
			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master/alignments/internal/subfamily"><strong>subfamily</strong></a>’
			alignments constructed to represent proposed
			homologies between representative members of <i>Parvoviridae</i> subfamilies</li>

			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master/alignments/internal/cross-genus"><strong>cross-genus</strong></a>’
			alignments constructed to represent proposed
			homologies between representative members of 'minor' <i>Parvoviridae</i> lineages</li>

			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master/alignments/internal/genus/"><strong>genus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific parvovirus genera.</li>
		  </ol>
         
 		</p>



	   <br>
	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees</strong>
	   </h3>
	   <hr>


		<p>

          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
        
		  <ol>
			<li> <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master/trees/family-root"><strong>Family-level (root)</strong></a> phylogeny (Rep)</li>
			<li> <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master/trees/genus-level"><strong>Genus-level</strong></a> phylogenies</li>
		  </ol>

 		</p>






	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/">Parvovirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



