<!DOCTYPE html>
<html lang="en-us">

  <head>
  
    <meta charset="UTF-8">
    <title>Parvoviridae-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="website/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="website/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="website/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Parvoviridae-GLUE</h1>
      <h2 class="project-tagline">A sequence data resource for parvoviruses (family <i>Parvoviridae</i>).</h2>
      
      <a href="./website/viruses.html" class="btn">Viruses</a>
      <a href="./website/paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="./website/tutorial.html" class="btn">Tutorial</a>

      <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/tree/master" class="btn">View on GitHub</a>
      
    </section>

    <section class="main-content">

	   
	   <p>

		 Access to this GitHub repository is currently restricted. To request access please contact:

		 <p>Robert J. Gifford (<a href="mailto:robert.gifford@glasgow.ac.uk">robert.gifford@glasgow.ac.uk</a>)</p>
		 
	   </p>



	   <h3>
		 <a id="description" class="anchor" href="#description" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Description</strong>
	   </h3>
	   <hr>

		 <p>
		 
		   This is <b>Parvoviridae-GLUE</b>, a GLUE
		   project designed to support comparative genomic and evolutionary analysis of
		   parvoviruses
		   (family <a target="_blank" href="https://www.sciencedirect.com/science/article/pii/S0065352718300575"><i><b>Parvoviridae</b></i></a>).

		 </p>

 
		 <p>
		 
		   <b>Parvoviridae-GLUE</b> contains aligned, annotated genome sequence data for
		   <a href="./website/project.html"><b>parvoviruses</b></a>
		   and
		   <a href="./website/paleovirology.html"><b>endogenous viral elements (EVEs)</b></a>
		   derived from these viruses.
		   
		 </p>

        
         <p><img src="./website/images/parvoviridae-glue-header.png" alt="CRESS DNA virus phylogeny" /></p>
         

        
		 <br>

<blockquote>
		 <p>
		   The
		   <a target="_blank" href="https://evolution-outreach.biomedcentral.com/articles/10.1007/s12052-010-0273-6"><b>phylogenetic tree</b></a>
		   shown above, taken from a report by
		   <a target="_blank" href="https://dx.doi.org/10.1007/s00705-020-04632-4"><b>Pénzes <i>et al.</i> (2020)</b></a>,	    
		   shows the evolutionary relationships between currently recognised genera in  
		   the <b>family <i>Parvoviridae</i></b>. The tree was reconstructed from
		   an alignment of the highly-conserved tripartite helicase domain (167 aa) of the 
		   <a target="_blank" href="https://www.uniprot.org/uniprot/Q6TV13"><b>non-structural (NS)</b></a>
		   protein.
		 </p>
         
</blockquote>
		 

       <br>
	   <h3>
	   <a id="WhatGlueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>What is a GLUE project?</strong>
	   </h3>
       <hr>


	   <p>
		 <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/"><b>GLUE</b></a> is an open, integrated
		 software toolkit that provides functionality for storage and interpretation of
		 sequence data. 
	   </p>

	   <p>
		 <b>GLUE</b> supports the development of <b>“projects”</b> containing the data items
		 required for comparative genomic analysis 
		 (e.g.  sequences, multiple sequence alignments, genome feature annotations,
		 and other sequence-associated data). 
	   </p>

	   <p>
		 Projects are loaded into the <b>GLUE "engine"</b>, creating a relational database
		 that represents the semantic relationships between data items.
		 This provides a robust foundation for the implementation of <b>systematic
		 comparative analyses</b> and the development of <b>sequence-based resources</b>. 
	   </p>

	   <p>		 
		  The core schema of this database can be extended to accommodate
		  the idiosyncrasies of different projects, and
		  GLUE provides a scripting layer (based on JavaScript) for developing custom
		  analysis tools. 
	   </p>

	   <p>
		  Hosting of GLUE projects in an online version control system (e.g. GitHub) provides
		  a mechanism for their stable, collaborative development.
	   </p>
	   
	   

		   
		 <p>
		   Some examples of '<b>sequence-based resources</b>' built for viruses using GLUE include:

		 </p>
		   
		   

		 <p>

		 <ul>
		 		 
		   <li> 
			 <a target="_blank" href="http://cov-glue.cvr.gla.ac.uk/"><b>COV-GLUE</b></a>: 
			 A GLUE resource for tracking genetic variation in SARS-COV2.
			 CoV-GLUE contains a database of amino acid replacements, insertions and
			 deletions which have been observed in GISAID hCoV-19 sequences sampled from the pandemic
		   </li>
		   <br>
		   
		   <li>
			 <a target="_blank" href="http://rabv-glue.cvr.gla.ac.uk/"><b>RABV-GLUE</b></a>:
			 Tailored toward epidemiological tracking of rabies virus (RABV).
			 Includes a database of RABV sequences and metadata from NCBI, updated daily and arranged into major and minor clades, and
			 an analysis tool providing genotyping, analysis and visualisation of submitted FASTA sequences.
 		   </li>
 		   <br>
		   
		   <li>
			 <a target="_blank" href="http://hcv-glue.cvr.gla.ac.uk/"><b>HCV-GLUE</b></a>:
			 This GLUE resource aims to support analysis of drug resistance and vaccine
			 escape in hepatitis C virus (HCV).
			 A database of HCV sequences and metadata from NCBI, updated daily and arranged
			 into clades (genotypes, subtypes). As well as pre-built multiple-sequence
			 alignments of NCBI sequences, it includes an analysis tool providing genotyping,
			 drug resistance analysis and visualisation of submitted FASTA sequences.
           </li>
	   
		 </ul>

		   
		 </p>
	   


	   <br>
	   <h3>
	   <a id="WhyGlueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>What does building the Parvoviridae-GLUE project offer?</strong>
	   </h3>
	   <hr>

		 
 
	   <p>
		 
		  Parvoviridae-GLUE offers a number of advantages for performing comparative sequence
		  analysis of parvoviruses:

		 <ol>

		   <li> <strong>Reproducibility</strong>. 
		   For many reasons, bioinformatics analyses are notoriously difficult to reproduce.
		   The GLUE framework supports the implementation of fully reproducible 
		   comparative genomics through the introduction of data standards and the use 
		   of a relational database to capture the semantic links between data items.
		   </li>

		   <li> <strong>Validation</strong>. 
		   Building GLUE projects entails mapping the semantic links between data items
		   (e.g. sequences, tabular data, multiple sequence alignments).
		   This process provides an opportunity 
		   for cross-validation, and thereby enforces a high level of data integrity.
		   </li>
		   		   
		   <li> <strong>Reusable data objects and analysis logic</strong>. 
		   For many - if not most - comparative genomic analyses, data preparation is nine
		   tenths of the battle. The GLUE framework has been designed to ensure that 
		   work spent preparing high-value data items such as multiple sequence alignments
		   need only be performed once. Hosting of GLUE projects in an online version control 
		   system such as GitHub allows for collaborative management of important data items
		   and community testing of hypotheses.
		   </li>

		   <li> <strong>Standardisation of the genomic co-ordinate space</strong>. GLUE
		   projects allow all sequences to utilise the coordinate space of a chosen 
		   reference sequence. Contingencies associated with insertions and deletions
		   (indels) are handled in a systematic way.
		   </li>
		   
		   <li> <strong>Predefined, fully annotated reference sequences</strong>:
		   This project includes fully-annotated reference sequences for major lineages
		   within the <i>Parvoviridae</i> family.
		   </li>
		   
		   <li> <strong>Alignment trees</strong>:
		   GLUE allows linking of alignments constructed at distinct taxonomic levels
		   via an "alignment tree" data structure.
		   </li>


		 </ol>

	   </p>


  
       <br>
	   <h3>
	   <a id="glueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Building this GLUE project</strong>
	   </h3>
       <hr>


      	   
	   <p>
	
         On computers with the GLUE software framework
         <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/installation"><b>installed</b></a>,
         the <b>Parvoviridae-GLUE</b> project can be instantiated by navigating to the project folder,
         initiating GLUE, and issuing the following command in the GLUE shell:
         
  <pre>
  <code>Mode path: /
  GLUE> run file buildCoreProject.glue</code></pre>

         This will build the Parvoviridae-GLUE core project by executing the commands in
		 <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/buildCoreProject.glue"><b>this file</b></a>.
		 
	   </p>

	   <p>
		 The core project comprises a  
		 dataset designed to represent the phylum in a minimal way (i.e. by including 
		 only one or a handful of annotated reference sequences for each major lineage). 
	   </p>


	   <p>
		 We have also created genus-level, extension projects individual genera within the
		 subfamily <i>Parvovirinae</i>.
		 These extensions incorporate a more extensive set of species for this virus family.
		 Once the core project has been built, the parvoviridae extension project can be
		 constructed, by executing the commands in
		 <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/buildGenusLevelProjects.glue"><b>this file</b></a>,
		 as follows.
         
  <pre>
  <code>Mode path: /
  GLUE> run file buildGenusLevelProjects.glue</code></pre>
		 		  
	   </p>


	   <p>
		 The <i>Parvoviridae</i> extension projects can be further extended to incorporate
		 EVE sequences by executing the commands in
		 <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/buildGenusLevelPaleoProjects.glue"><b>this file</b></a>,
		 as follows.
         
  <pre>
  <code>Mode path: /
  GLUE> run file buildGenusLevelPaleoProjects.glue</code></pre>
		 		  
	   </p>

	   <p>
		 The <i>Parvoviridae</i> paleovirus extension incorporates a set of endogenous
		 viral elements (EVEs) recovered from the genomes of metazoan species. Building
		 the paleovirus extension allows automated alignment and phylogeny reconstruction
		 for individual EPV lineages in the project, based on the classifications
		 in
		 <a target="blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/eve/"><b>these files</b></a>.
		 Individual EPV sequences have been classified into sets considered likely to 
		 have arisen from the same germline colonisation event. Loci have been named using
		 a systematic approach (see 
		 <a href="./website/paleoviruses.html"><b>here</b></a> for details). 
		 
	   </p>
       <br>



	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>


		  <p>
		

		  Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		  <b>(2018)</b>
		  <br>
		  GLUE: A flexible software system for virus sequence data.
		  <br>
		  <i>BMC Bioinformatics</i>
		  <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		  <br>
		  <br>

		  Zhu H, Dennis T, Hughes J, and RJ Gifford
		  <b>(2018)</b>
		  <br>
		  Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		  <a href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		  <br>
		  <br>

		  Gifford RJ, Blomberg B, Coffin JM, Fan H, Heidmann T, Mayer J, Stoye J, Tristem M, and WE Johnson
		  <b>(2018)</b>
		  <br>
		  Nomenclature for endogenous retrovirus (ERV) loci.
		  <br>
		  <i>Retrovirology</i>
		  <a href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1" target="_blank">[view]</a> 
		  <br>
		  <br>

		  Katzourakis A. and RJ. Gifford
		  <b>(2010)</b>
		  <br>
		  Endogenous viral elements in animal genomes.
		  <br>
		  <i>PLoS Genetics</i>
		  <a href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1001191" target="_blank">[view]</a>

		  </p>





      
	   <br>

	   <h3>
		 <a id="contributors" class="anchor" href="#contributors" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Contributors</strong>
	   </h3>
	   <hr>

	   
		 <p>
			<a target="_blank" href="https://giffordlabcvr.github.io/virus-evolution/index.html">Robert J. Gifford</a> (<a href="mailto:robert.gifford@glasgow.ac.uk">robert.gifford@glasgow.ac.uk</a>)	 
		 </p>
		 
		 <p>
			Mac Campbell <!-- >(<a href="mailto:>"></a>) -->
		 </p>
		 
		 <p>Shannon Loncar</p>
		 <p>Robert Kotin</p>


	   <br>
	   <h3>
		 <a id="license" class="anchor" href="#license" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>License</strong>
	   </h3>
       <hr>


		 <p>
		   This project is licensed under the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License v. 3.0</a>.
		 </p>

	   <br>






	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/DIGS-tool">Parvoviridae-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>
鍟 