<!DOCTYPE html>
<html lang="en-us">

  <head>
  
    <meta charset="UTF-8">
    <title>Parvovirus-GLUE by giffordlabcvr: Overview</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="website/assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="website/assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="website/assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Parvovirus-GLUE</h1>
      
      <h2 class="project-tagline">Resources for comparative genomic analysis of parvoviruses.</h2>
            
      <a href="./website/html/viruses.html" class="btn">Viruses</a>
      <a href="./website/html/paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="./website/html/highlights.html" class="btn">Highlights</a>
      <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/tree/master" class="btn">View on GitHub</a>
      
    </section>

    <section class="main-content">


	   <h3>
		 <a id="background" class="anchor" href="#description" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Comparative genomic analysis of parvoviruses using Parvovirus-GLUE</strong>
	   </h3>
	   <hr>


		 <p>		 
		   This is <b>Parvovirus-GLUE</b>, a GLUE project designed to support comparative
		   genomic and evolutionary analysis of <b>parvoviruses</b> 
           (family <a target="_blank" href="https://www.sciencedirect.com/science/article/pii/S0065352718300575"><i><b>Parvoviridae</b></i></a>).	   	 
		 </p>


		 <p>
           The parvoviruses are a diverse group of small, non-enveloped DNA viruses that
           infect an extremely broad and phylogenetically diverse range
           of animal species.  The parvovirus family includes numerous <b>pathogens of humans and domesticated animals</b>.
		 </p>


		 <p>
           However, parvoviruses are also being developed as <b>next-generation therapeutic tools</b>
           - for example, rodent protoparvoviruses are promising anticancer agents that show <b>natural
           oncotropism and oncolytic properties</b>, while adeno-associated viruses (AAVs) have been
           successfully adapted as <b>gene therapy vectors</b> and are leading candidates for the
           further development of human gene therapy.

		 </p>

	
		 <p><img src="./website/assets/images/parvovirus-glue-header.jpg" alt="Parvoviruses"/></p>
		 
<blockquote>
		 <p>
		   Comparative genomic analysis of parvoviruses can help reveal the biological basis
		   of parvovirus replication.
		   The insights obtained via comparative analysis of parvovirus genomes can directly
		   inform efforts to limit parvovirus-associated disease and develop parvovirus-based 
		   therapeutics. 
		 </p>
         
</blockquote>


         <br>
		 <p>
		   Since the emergence of the SARS-COV2 pandemic, many have become familiar with 
		   the use of <b>virus genome data</b>  to track the spread and evolution of pathogenic viruses
		   - e.g. via tools such as <a target="_blank" href="https://nextstrain.org/ncov/gisaid/global""><b>NextStrain</b></a>. 
		   However, it is less widely appreciated that the same kinds of data sets and comparative genomic approaches
		   can also be used to explore the structural and functional basis of virus adaptations. 
		 </p>

	   
		 <p>	   
		   The 
		   <a target="_blank" href="https://github.com/giffordlabcvr/gluetools"><b>GLUE software framework</b></a>
		   provides an extensible platform for implementing computational genomic
		   analysis of viruses in an <b>efficient, standardised and reproducible</b> way.
		   <b>GLUE projects</b> can not only incorporate all of the data items typically used in
		   comparative genomic analysis
		   (e.g. sequences, alignments, genome feature annotations) but can also represent the complex
		   semantic links between these data items via a <b>relational database</b>.
		   This 'poises' sequences and associated data for application in computational
		   analysis, minimising the requirement for labour-intensive pre-processing of datasets.
		 </p>      

		 <p>

		   GLUE projects are equally suited for carrying out exploratory work 
		   (e.g. using virus genome data to investigate structural and functional properties of viruses)
		   as they are for implementing operational procedures (e.g. producing
		   <a target="_blank" href="http://hcv-glue.cvr.gla.ac.uk/exampleReports/resistanceGeno1.html?hcvGlueVersion=0.1.68""><b>standardised reports</b></a>
		   in a public or animal health setting).
		 </p>

		 <p>
			Hosting of GLUE projects in an online version control system (e.g. <b>GitHub</b>) provides
			a mechanism for their stable, collaborative development, as shown below.
		 </p>
	  
		
		 <p><img src="./website/assets/images/github-hosting.jpg" alt="GitHub hosting figure"/></p>



		 <h3>
		 <a id="WhatGlueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>What is a GLUE project?</strong>
		 </h3>
		 <hr>


		 <p>
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/"><b>GLUE</b></a> is an open, integrated
		   software toolkit that provides functionality for storage and interpretation of
		   sequence data. 
		 </p>

		 <p>
		   <b>GLUE</b> supports the development of <b>“projects”</b> containing the data items
		   required for comparative genomic analysis 
		   (e.g.  sequences, multiple sequence alignments, genome feature annotations,
		   and other sequence-associated data). 
		 </p>

		
		 <p><img src="./website/assets/images/parvovirus-glue-project.jpg" alt="Parvovirus GLUE project figure"/></p>
		 

		 <p>
		   Projects are loaded into the <b>GLUE "engine"</b>, creating a relational database
		   that represents the semantic relationships between data items.
		   This provides a robust foundation for the implementation of <b>systematic
		   comparative analyses</b> and the development of <b>sequence-based resources</b>. 
		 </p>

		
		 <p><img src="./website/assets/images/parvovirus-glue-usage.jpg" alt="Using Parvovirus GLUE"/></p>
		 


		 <p>		 
			The core schema of this database can be extended to accommodate
			the idiosyncrasies of different projects, and
			GLUE provides a scripting layer (based on JavaScript) for developing custom
			analysis tools. 
		 </p>

		 <p>
		   Some examples of '<b>sequence-based resources</b>' built for viruses using GLUE include:

		 </p>
		 
		 

		 <p>

		 <ul>
			   
		   <li> 
			 <a target="_blank" href="http://cov-glue.cvr.gla.ac.uk/"><b>COV-GLUE</b></a>: 
			 A GLUE resource for tracking genetic variation in SARS-COV2.
			 CoV-GLUE contains a database of amino acid replacements, insertions and
			 deletions which have been observed in GISAID hCoV-19 sequences sampled from the pandemic
		   </li>
		   <br>
		 
		   <li>
			 <a target="_blank" href="http://rabv-glue.cvr.gla.ac.uk/"><b>RABV-GLUE</b></a>:
			 Tailored toward epidemiological tracking of rabies virus (RABV).
			 Includes a database of RABV sequences and metadata from NCBI, updated daily and arranged into major and minor clades, and
			 an analysis tool providing genotyping, analysis and visualisation of submitted FASTA sequences.
		   </li>
		   <br>
		 
		   <li>
			 <a target="_blank" href="http://hcv-glue.cvr.gla.ac.uk/"><b>HCV-GLUE</b></a>:
			 This GLUE resource aims to support analysis of drug resistance and vaccine
			 escape in hepatitis C virus (HCV).
			 A database of HCV sequences and metadata from NCBI, updated daily and arranged
			 into clades (genotypes, subtypes). As well as pre-built multiple-sequence
			 alignments of NCBI sequences, it includes an analysis tool providing genotyping,
			 drug resistance analysis and visualisation of submitted FASTA sequences.
		   </li>
	 
		 </ul>

		 
		 </p>
	   


		 <br>
		 <h3>
		 <a id="WhyGlueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>What does building the Parvovirus-GLUE project offer?</strong>
		 </h3>
		 <hr>


 
	   <p>
		 
		  Parvovirus-GLUE offers a number of advantages for performing comparative sequence
		  analysis of parvoviruses:

		  <ol>

			<li> <strong>Reproducibility</strong>. 
			For many reasons, bioinformatics analyses are notoriously difficult to reproduce.
			The GLUE framework supports the implementation of fully reproducible 
			comparative genomics through the introduction of data standards and the use 
			of a relational database to capture the semantic links between data items.
			</li>
			<br>
		  
			<li> <strong>Reusable data objects and analysis logic</strong>. 
			For many - if not most - comparative genomic analyses, data preparation is nine
			tenths of the battle. The GLUE framework has been designed to ensure that 
			work spent preparing high-value data items such as multiple sequence alignments
			need only be performed once. Hosting of GLUE projects in an online version control 
			system such as GitHub allows for collaborative management of important data items
			and community testing of hypotheses.
			</li>
			<br>


			<li> <strong>Validation</strong>. 
			Building GLUE projects entails mapping the semantic links between data items
			(e.g. sequences, tabular data, multiple sequence alignments).
			This process provides an opportunity 
			for cross-validation, and thereby enforces a high level of data integrity.
			</li>
			<br>
		
			<li> <strong>Standardisation of the genomic co-ordinate space</strong>. GLUE
			projects allow all sequences to utilise the coordinate space of a chosen 
			reference sequence. Contingencies associated with insertions and deletions
			(indels) are handled in a systematic way.
			</li>
			<br>
		  
			<li> <strong>Predefined, fully annotated reference sequences</strong>:
			This project includes fully-annotated reference sequences for major lineages
			within the <i>Hepadnaviridae</i> family.
			</li>
			<br>
		  
			<li> <strong>Alignment trees</strong>:
			GLUE allows linking of alignments constructed at distinct taxonomic levels
			via an ""<b>alignment tree</b>"
			data structure. In the alignment tree, each alignment
			is constrained to a standard reference sequence, thus all multiple sequence
			alignments are linked to one another via a standardised coordinate system.
			</li>
			
		  </ol>
		  
		  
	   </p>


  
       <br>
	   <h3>
	   <a id="glueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Building this GLUE project</strong>
	   </h3>
       <hr>


      	   
	   <p>
	
         On computers with the GLUE software framework
         <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/installation"><b>installed</b></a>,
         the <b>Parvovirus-GLUE</b> project can be instantiated by navigating to the project folder,
         initiating GLUE, and issuing the following command in the GLUE shell:
         
  <pre>
  <code>Mode path: /
  GLUE> run file buildCoreProject.glue</code></pre>

         This will build the Parvovirus-GLUE core project by executing the commands in
		 <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/buildCoreProject.glue"><b>this file</b></a>.
		 
	   </p>

	   <p>
		 The core project comprises a  
		 dataset designed to represent the phylum in a minimal way (i.e. by including 
		 only one or a handful of annotated reference sequences for each major lineage). 
	   </p>


	   <p>
		 We have also created genus-level, extension projects individual genera within the
		 subfamily <i>Parvovirinae</i>.
		 These extensions incorporate a more extensive set of species for this virus family.
		 Once the core project has been built, the parvoviridae extension project can be
		 constructed, by executing the commands in
		 <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/buildGenusLevelProjects.glue"><b>this file</b></a>,
		 as follows.
         
  <pre>
  <code>Mode path: /
  GLUE> run file buildGenusLevelProjects.glue</code></pre>
		 		  
	   </p>


	   <p>
		 The <i>Parvoviridae</i> genus-level extension projects can be further extended to incorporate
		 EVE sequences by executing the commands in
		 <a href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/buildGenusLevelPaleoProjects.glue"><b>this file</b></a>,
		 as follows.
         
  <pre>
  <code>Mode path: /
  GLUE> run file buildGenusLevelPaleoProjects.glue</code></pre>
		 		  
	   </p>

	   <p>
		 The <i>Parvoviridae</i> paleovirus extension incorporates a set of endogenous
		 viral elements (EVEs) recovered from the genomes of metazoan species. Building
		 the paleovirus extension allows automated alignment and phylogeny reconstruction
		 for individual EPV lineages in the project, following the classifications outlined in
		 <a target="blank" href="https://github.com/giffordlabcvr/Parvovirus-GLUE/blob/master/tabular/eve/"><b>these files</b></a>.
		 Individual EPV sequences have been classified into sets considered likely to 
		 have arisen from the same germline colonisation event. Loci have been named using
		 a systematic approach (see 
		 <a href="./website/html/paleoviruses.html"><b>here</b></a> for details). 
		 
	   </p>
       <br>


	   <h3>
		 <a id="contributors" class="anchor" href="#contributors" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Contributors</strong>
	   </h3>
	   <hr>

	   
		 <p>
			<a target="_blank" href="https://giffordlabcvr.github.io/virus-evolution/index.html">Robert J. Gifford</a> (<a href="mailto:robert.gifford@glasgow.ac.uk">robert.gifford@glasgow.ac.uk</a>)	 
		 </p>
	   
		 <p>
			<a target="_blank" href="https://giffordlabcvr.github.io/virus-evolution/index.html">Mac Campbell</a> (<a href="mailto:macampbell2@alaska.edu">robert.gifford@glasgow.ac.uk</a>)	 
		 </p>
		 
 

	   <br>





	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>


		  <p>
		
  
		  Campbell M, Loncar S, Gifford RJ, Kotin R, and RJ Gifford
		  <b>(2021)</b>
		  <br>
		  Comparative analysis reveals the long-term co-evolutionary history of parvoviruses and vertebrates.
		  <br>
		  <a href="https://www.biorxiv.org/content/10.1101/2021.10.25.465781v1">[preprint]</a> 
		  <br>
		  <br>

		  Hildebrandt E, Penzes J, Gifford RJ, Agbandje-Mckenna M, and R Kotin
		  <b>(2020)</b>
		  <br>
		  Evolution of dependoparvoviruses across geological timescales – implications for design of AAV-based gene therapy vectors.
		  <br>
		  <i>Virus Evolution</i> 
		  <a href="https://academic.oup.com/ve/article/doi/10.1093/ve/veaa043/5842153">[view]</a> 
		  <br>
		  <br>

		  Pénzes JJ, de Souza WM, Agbandje-Mckenna M, and RJ Gifford
		  <b>(2019)</b>
		  <br>
		  An ancient lineage of highly divergent parvoviruses infects both vertebrate and invertebrate hosts.
		  <br>
		  <i>Viruses</i>
		  <a href="https://www.mdpi.com/1999-4915/11/6/525" target="_blank">[view]</a> 
		  <br>
		  <br>

		  Callaway HM, Subramanian S, Urbina C, Barnard K, Dick R, Hafentein SL, Gifford RJ, and CR Parrish
		  <b>(2019)</b>
		  <br>
		  Examination and reconstruction of three ancient endogenous parvovirus capsid proteins in rodent genomes.
		  <br>
		  <i>Journal of Virology</i>
		  <a href="https://jvi.asm.org/content/93/6/e01542-18.long" target="_blank">[view]</a>
		  <br>
		  <br>

		  Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		  <b>(2018)</b>
		  <br>
		  GLUE: A flexible software system for virus sequence data.
		  <br>
		  <i>BMC Bioinformatics</i>
		  <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		  <br>
		  <br>

		  Pénzes JJ, Marsile-Medun S, Agbandje-McKenna M, and RJ Gifford
		  <b>(2018)</b>
		  <br>
		  Endogenous amdoparvovirus-related elements reveal insights into the biology and evolution of vertebrate parvoviruses.
		  <br>
		  <i>Virus Evolution</i>
		  <a href="https://academic.oup.com/ve/article/4/2/vey026/5179240" target="_blank">[view]</a> 
		  <br>
		  <br>

		  de Souza WM, Tristan TPW, Fumagalli MJ, Araujo J, Sabino-Santos Jr. G, Maia FGM, Acrani GO, Carrasco AOT, Romeiro MF, Modha S, Vieira LC, Ometto TL, Queiroz LH, Durigon EL, Nunes MRT, Figueiredo LTM, and RJ Gifford 
		  <b>(2018)</b>
		  <br>
		  Whole-genome sequencing of parvoviruses from wild and domestic animals in Brazil provides new insights into parvovirus distribution and diversity.
		  <br>
		  <i>Viruses</i>
		  <a href="http://www.mdpi.com/1999-4915/10/4/143" target="_blank">[view]</a> 

		  <br>
		  <br>
		  Souza WM, Romeiro MF, Fumagalli MJ, Modha S, de Araujo J, Queiroz LH, Durigon EL, Figueiredo LT, Murcia PR, Gifford RJ.
		  <b>(2017)</b>
		  <br>
		  Chapparvoviruses occur in at least three vertebrate classes and have a broad biogeographic distribution.
		  <br>
		  <i>J Gen Virol.</i>
		  <a href="http://jgv.microbiologyresearch.org/content/journal/jgv/10.1099/jgv.0.000671" target="_blank">[view]</a>
 
 		 </p>

     
	   <br>



	   <h3>
		 <a id="license" class="anchor" href="#license" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>License</strong>
	   </h3>
       <hr>


		 <p>
		   This project is licensed under the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License v. 3.0</a>.
		 </p>

	   <br>






	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/DIGS-tool">Parvovirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>

