<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Parvoviridae-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">

  </head>


  <body>
    <section class="page-header">
      <h1 class="project-name">Parvoviridae-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of parvoviruses.</h2>

      <a href="../index.html" class="btn">Home</a>
      <a href="./viruses.html" class="btn">Viruses</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>



    <section class="main-content">


	   <h3>
		 <a id="'Highlights" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Hepadnaviridae-GLUE: Highlights</strong>
	   </h3>
	   <hr>

		<p>
          These highlights pages aim to provide a brief overview of selected data items
          contained within the <b>Parvoviridae-GLUE</b> project, and to illustrate
          how it can be used to implement reproducible comparative analyses. 
 		</p>



	   <h3>
		 <a id="Dependo.43.Octodon" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Dependo.6-Elephas and Dependo.43-Octodon</strong>
	   </h3>
	   <hr>


		<p>
          Amongst the various EPV that have been reported so far, several have been found
          to encode replicase genes with <b>intact open reading frames</b>.
          In a previous study, we investigated one of these genes - found in the genome of the degu -
          a South American rodent.
          Intriguingly, we found that it showed a clear, tissue-specific pattern of activity  
        </p>

          
 
		<p>

          Furthermore, intact replicase genes in independently acquired
          dependoparvovirus-derived EPV in rodents and afrotherians have been shown to
          exhibit similar patterns of tissue-specific expression.
        </p>


<blockquote>
		<p>
		  Ichthama.2.Serpentes - left to right:
		</p>
         
</blockquote>


          
 
		<p>
          Tissue-specific transcription of a completely intact, replicase-encoding EPV,
          present in the genome of the degu (Octodon degus)
        </p>

  
  
  
          
		<p>
           Related data items: 
		  <ol>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/eve/parvovirinae/epv-amdo-refseqs-side-data.tsv">Sphenodon ortholog</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/eve/parvovirinae/epv-erythyro-refseqs-side-data.tsv">Avian consensus</a></li>
		  </ol>

  		</p>




	   <h3>
		 <a id="Amdo.1.Ellobius" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Amdo.1.Ellobius</strong>
	   </h3>
	   <hr>


		<p>
           The Transcaucasian mole vole (E. lutescens) is a species of cricetid rodent
           inhabiting semi-arid or grassland areas in Central Asia, and is notable for
           its unusual karyotype - all individuals possessing a diploid number of 17
           chromosomes.
        </p>

          
 
		<p>
          
           Only a single sex chromosome is present - with the Y chromosome having been
           eliminated.
           This interesting characteristic has motivated the sequencing of the E.lutescens
           genome, as well as that of a sister species - the northern mole vole
           (E. talpinus). We identified the corresponding empty genomic integration sites
           in E. talpinus - indicating that both elements were incorporated into the
           E.lutescens germline within the last 10 million years.
        </p>

          
          
 
		<p>
           We identified four further EPV in mammal and reptile genomes that are intermediate
           between amdoparvoviruses and their sister genus (Protoparvovirus) in terms of
           their phylogenetic placement and genomic features. In particular, we identify
           a genome-length EPV in the genome of a pit viper (Protobothrops mucrosquamatus)
           that is more like a protoparvovirus than an amdoparvovirus in terms of its
           phylogenetic placement and the structural features of its capsid protein (as
           revealed by homology modeling), yet exhibits characteristically amdoparvovirus-like
           genome features including: (i) a putative middle ORF gene; (ii) a capsid gene
           that lacks a phospholipase A2 (PLA2) domain; (iii) a genome structure consistent
           with an amdoparvovirus-like mechanism of capsid gene expression.
   		</p>

		<p>
           Related data items: 
		  <ol>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/eve/parvovirinae/epv-amdo-refseqs-side-data.tsv">Sphenodon ortholog</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/eve/parvovirinae/epv-erythyro-refseqs-side-data.tsv">Avian consensus</a></li>
		  </ol>

  		</p>




	   <h3>
		 <a id="Dependo.54.Cavia" class="anchor" href="#Dependo.54.Cavia" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Dependo.54.Cavia</strong>
	   </h3>
	   <hr>


		<p>

          enREP is one of several dependoparvovirus-derived sequences we identified in the
          Cavia germline. Since many of these elements span different genomic regions,
          it is likely that they derive from a smaller number of initial integrants,
          including at least one genome-length or near genome-length sequence, which
          latterly became fragmented (e.g. by transposable element insertions, chromosome
          re-arrangements). We identified two groups of elements that spanned the rep gene.
          The first includes enRep sequences from both guinea pig species examined.
          The second includes a longer element that spans an entire rep gene. This second
          group of elements is much older, being shared across rodents species that
          diverged >70 million years ago (Fig 1).
          
        </p>

          
          
 
		<p>


          While most of this EPV sequence is degraded, the portions included in the
          enRep-Myo9 gene are intact in multiple species of guinea pig (genus Cavia),
          consistent with evolution under purifying selection.
        </p>

          
          
 
		<p>
          The broad expression of enRep-Myo9 mRNA, the conservation of itâ€™s EPV-derived
          regions in multiple species of guinea pig (genus Cavia), indicate that this
          host-virus fusion gene encodes a protein with a physiologically relevant role.
        </p>

          
          
 
		<p>

          The viral portions of enRep-Myo9 derive from an ancient dependoparvovirus
          (genus Dependoparvovirus)  that was incorporated into the genome of caviomorph
          rodents >6 million years ago.
          
        </p>

		<p>
           Related data items: 
		  <ol>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/eve/parvovirinae/epv-amdo-refseqs-side-data.tsv">EPV-Dependo.54-Cavia sequence</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/eve/parvovirinae/epv-erythyro-refseqs-side-data.tsv">Fusion product (AA)</a></li>
		  </ol>

  		</p>




	   <h3>
		 <a id="Ichthama.2.Serpentes" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Ichthama.2.Serpentes</strong>
	   </h3>
	   <hr>


		<p>
           <i>Ichthamaparvovirus</i> is a recently discovered genus subfamily <i>Hamaparvovirinae</i>.
           EPVs derived from this genus are rare, but have been detected in
           (i) the tiger-tail seahorse, and (ii) various species of sea snake.
 
  		</p>

 
		<p>         
           This sequence showed a close relationship to ScChPV - the prototype member of the 
           <i>Ichthamaparvovirus</i> genus.
   		</p>




        <p><img src="" alt="Amdo.1.Ellobius" /></p>
 		
	
		<p>
           Related data items: 
		  <ol>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/eve/parvovirinae/epv-amdo-refseqs-side-data.tsv">Zipped orthologs (fasta)</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/eve/parvovirinae/epv-erythyro-refseqs-side-data.tsv">Ichthama.2.Serpentes consensus</a></li>
		  </ol>

  		</p>




	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/">Parvoviridae-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



