<!DOCTYPE html>
<html lang="en-us">

  <head>
  
    <!-- Global site tag (gtag.js) - Google Analytics -->
	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-127420264-1"></script>
	<script>
	  window.dataLayer = window.dataLayer || [];
	  function gtag(){dataLayer.push(arguments);}
	  gtag('js', new Date());
	  gtag('config', 'UA-127420264-1');
	</script>


    <meta charset="UTF-8">
    <title>Parvoviridae-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Parvoviridae-GLUE</h1>
      <h2 class="project-tagline">Paleovirology resources for Parvoviridae DNA viruses.</h2>
      <a href="../index.html" class="btn">Home</a>
      <a href="./project.html" class="btn">Viruses</a>
      <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/zipball/master" class="btn">Download .zip</a>
      <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>

    <section class="main-content">


	   <h3>
		 <a id="tutorialOverview" class="anchor" href="#tutorialOverview" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>How to use Parvoviridae-GLUE - the paleovirus fossil record</strong>
	   </h3>


	   <p>
         
         This <b>tutorial</b> comprises the following steps:

		 <ol>
		   <li> <b>Exporting</b> EVE sequences and data from Parvoviridae-GLUE.</li>
		   <li> <b>Extracting</b> isolate data from GenBank files.</li>
		   <li> <b>Filtering</b> sequence data to obtain a valid analysis set.</li>
		   <li> <b>Defining virus reference sequences</b> and associating them with annotations.</li>
		   <li> <b>Constructing an alignment</b> using a codon-aware method.</li>
		   <li> <b>Mapping feature coverage</b> of sequence members within an alignment.</li>
		   <li> Reconstructing annotated, midpoint-rooted <b>phylogenetic trees</b>.</li>
		   <li> Using GLUE projects as a basis for <b>collaborative research and resource development</b>.</li>
		 </ol>


 	   </p>
 	   


	   <h3>
		 <a id="Background" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Background - the 'fossil record' of the <i>Parvoviridae</i></strong>
	   </h3>

        <p><img src="./images/parrots-header.png" alt="Parrots" /></p>

		<p>

We screened animal genomes to identify EPV that might shed light on the evolution of amdoparvoviruses. We identified six EPV that disclosed greater similarity to amdoparvovirus reference sequences than to those representing the sister genus (Protoparvovirus). Comparative analysis of these sequences revealed new insights into the evolutionary history of the Amdoparvovirus and Protoparvovirus genera.
		</p>


		<p>

The Transcaucasian mole vole (E. lutescens) is a species of cricetid rodent inhabiting semi-arid or grassland areas in Central Asia, and is notable for its unusual karyotype - all individuals possessing a diploid number of 17 chromosomes. Only a single sex chromosome is present - with the Y chromosome having been eliminated. This interesting characteristic has motivated the sequencing of the E.lutescens genome, as well as that of a sister species - the northern mole vole (E. talpinus). We identified the corresponding empty genomic integration sites in E. talpinus - indicating that both elements were incorporated into  the E.lutescens germline within the last 10 million years.
		</p>


		<p>

We identified four further EPV in mammal and reptile genomes that are intermediate between amdoparvoviruses and their sister genus (Protoparvovirus) in terms of their phylogenetic placement and genomic features. In particular, we identify a genome-length EPV in the genome of a pit viper (Protobothrops mucrosquamatus) that is more like a protoparvovirus than an amdoparvovirus in terms of its phylogenetic placement and the structural features of its capsid protein (as revealed by homology modeling), yet exhibits characteristically amdoparvovirus-like genome features including: (i) a putative middle ORF gene; (ii) a capsid gene that lacks a phospholipase A2 (PLA2) domain; (iii) a genome structure consistent with an amdoparvovirus-like mechanism of capsid gene expression.
		</p>


		<p>

The other EPB were identified in basal mammals - two afrotherian species - including the aardvark and the cape hyrax, and one marsupial - the Tasmanian devil. 
		</p>


		<p>

Taken together, these findings indicate that amdoparvovirus host range extends to rodents, and that parvovirus lineages possessing a mixture of proto- and amdoparvovirus-like characteristics have circulated in the past.

		</p>







	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/">Parvoviridae-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



