<!DOCTYPE html>
<html lang="en-us">

  <head>
  
    <meta charset="UTF-8">
    <title>Parvoviridae-GLUE by giffordlabcvr: An example-based tutorial - parvoviruses and the mink fur industry</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Parvoviridae-GLUE</h1>
      <h2 class="project-tagline">A GLUE project for Parvoviridae.</h2>
      <a href="../index.html" class="btn">Home</a>
      <a href="./project.html" class="btn">Viruses</a>
      <a href="./paleovirology.html" class="btn">Paleoviruses</a>
      <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/zipball/master" class="btn">Download .zip</a>
      <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>

    <section class="main-content">


	   <h3>
		 <a id="sequenceData" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>How to use Parvoviridae-GLUE - an example-driven tutorial</strong>
	   </h3>


	   <p>
         
         This tutorial focuses on <b>carnivore amdoparvovirus 1</b> - also known as
         
         <a target="_blank" href="https://en.wikipedia.org/wiki/Aleutian_disease"><b>Aleutian mink disease virus (AMDV)</b></a>.
         
         It comprises the following steps:
		 <ol>
		   <li> <b>Downloading</b> selected sequences from GenBank.</li>
		   <li> <b>Extracting</b> isolate data from GenBank files.</li>
		   <li> <b>Filtering</b> sequence data to obtain a valid analysis set.</li>
		   <li> <b>Defining virus reference sequences</b> and associating them with annotations.</li>
		   <li> <b>Constructing an alignment</b> using a codon-aware method.</li>
		   <li> <b>Mapping feature coverage</b> of sequence members within an alignment.</li>
		   <li> Reconstructing annotated, midpoint-rooted <b>phylogenetic trees</b>.</li>
		   <li> Using GLUE projects as a basis for <b>collaborative research and resource development</b>.</li>
		 </ol>


 	   </p>
 	   

	   <p>
         If you haven't decided yet whether or not you want to use GLUE, 
         you might find it useful simply to look through this tutorial, as it should
         provide useful insight into the kind of things that can be done with GLUE projects.
 	   </p>

	   <p>
         If you've already committed to using GLUE, you 
         may find it useful to also investigate the
         <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/exampleProject"><b>GLUE example project</b></a>
         when attempting this tutorial.
 	   </p>


	   <h3>
		 <a id="Background" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Background 1: Carnivores, parvoviruses and the fur industry</strong>
	   </h3>

        <p><img src="./images/fur-trade-parvoviruses-header.png" alt="Parvoviruses and the fur industry" /></p>


	   <p>
        Humans have established relatively close relationships with several carnivore species.
        But whereas some are kept as companion animals, others are hunted or farmed for meat or fur.
        In Europe, several 
		 <a target="_blank" href="https://en.wikipedia.org/wiki/Mustelidae">mustelid</a>      
        species have historically been hunted for fur, including
        European mink (<i>Mustela lutreola</i>),
		 <a target="_blank" href="https://en.wikipedia.org/wiki/Lady_with_an_Ermine">ermine</a>      
        (<i>Mustela ermina</i>)
        and European polecat (<i>Mustela putorius</i>).
        However, the fur of the    
		 <a target="_blank" href="https://animaldiversity.org/accounts/Neovison_vison/">North American mink </a>      
        (<i>Neovison vison</i>) is considered
        superior in quality to all of these.
 	   </p>

	   <p>
         Furthermore, while records of efforts to breed ermine (stoats) in captivity exist,
         these ventures have apparently been short-lived. By contrast, North American mink
         are extensively farmed. 
         The first mink farms were founded in the 1860s, in Upstate New York, and farming
         of American mink was introduced into Europe in the early 1930s. The breeding of
         various fur colour mutants led to a boom in the mink industry in the two decades
         that followed.  
 	   </p>

	   <p>
         One unintended consequence of the post-war boom in mink farming was the
         introduction of American mink into Europe (as escaped animals took up residence
         in local habitats). American mink (hereafter referred to simply as ‘mink’)
         are now a relatively widespread invasive species in Europe.         
 	   </p>

	   <p>

         In the 1930s a severe disease emerged in farmed mink. This disease was
         originally identified in the Aleutian mink breed and was consequently named
         <b>Aleutian disease (AD)</b>. However, it was soon discovered to afflict American mink
         in general.
         AD is caused by an parvovirus in genus (<i>Amdoparvovirus</i>)
         called <b>Aleutian mink disease virus (AMDV)</b>. This virus was originally considered
         a species in its own right, but has recently been reclassified as a sublineage of
         carnivore amdoparvovirus 1.
      
         AD is presently considered the most important infectious disease
         affecting farm-raised mink.
  	   </p>

	   <p>
 	   </p>


	   <h3>
		 <a id="Background" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Background 2: AMDV-related parvoviruses in wild versus farmed carnivores</strong>
	   </h3>

        <p><img src="./images/fur-trade-parvoviruses-midlevel.png" alt="Carnivores, parvoviruses and the fur trade" /></p>

	   <p>
         Infection with AMDV - or related amdoparvoviruses -
         is apparently widespread in wild mink as well as in farmed
         animals. In addition, related viruses have been identified in several other carnivore
         species, including gray foxes, skunks, raccoon dogs and red pandas. However,
         relatively little is known about the biology of amdoparvovirus infection in the
         natural environment, or the broader distribution of amdoparvovirus infections
         in wild species.
 	   </p>

	   <p>

         <u>Most importantly</u>, it is not clear whether the pathology of AD in captive mink is
         typical of disease that occurs in the wild, or if factors associated with fur
         farming somehow enabled the emergence of the disease. 
 	   </p>


	   <p>
         Increased availability
         of molecular sequence data mean it might now be feasible to gain some insight
         into the natural history and evolution of amdoparvoviruses, and this in turn
         may shed light on the emergence of AD.
 	   </p>


	   <p>
         In this tutorial we will use the <b>Parvoviridae-GLUE</b> project
         and published parvovirus sequence data to <b>(i)</b>
         investigate AMDV distribution diversity and evolution, and <b>(ii)</b> establish
         a sequence-based resource for AMDV.
 	   </p>


	   <h3>
		 <a id="DownloadFromNcbi" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>1. Downloading all available AMDV sequences from GenBank</strong>
	   </h3>


	   <p>
         To download all AMDV entries in NCBI GenBank, we 
         will use a customised version of GLUE's  module
         <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/moduleReference/moduleType/ncbiImporter">'<b>ncbiImporter</b>'</a>       
         module. Our project-specific configuration of the module can be viewed
		 <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/modules/genus/parvo/parvoviridaeNcbiImporterAmdv.xml"><b>here</b></a>.
         Viewing the file, you can probably see that it is configured to download sequences
         based on a query phrase:

	   <p>
        <i>"Carnivore amdoparvovirus 1"[Organism] AND 200:2000[SLEN]</i>
 	   </p>


	   <p>        
         This 'eSearchTerm' is a standard NCBI entrez text query that
         specifies all GenBank entries labelled "beak and feather disease" in the
         'Organism' field and between 200-2000 nucleotides (nt) in length.

 	   </p>

	   <p>
         To use the module, first initiate GLUE on the command line as follows: 
         
         
  <pre>
  <code>MyComputer:Parvoviridae-GLUE rob$ gluetools.sh 
  GLUE Version 1.1.103
  Copyright (C) 2015-2020 The University of Glasgow
  This program comes with ABSOLUTELY NO WARRANTY. This is free software, and you
  are welcome to redistribute it under certain conditions. For details see
  GNU Affero General Public License v3: http://www.gnu.org/licenses/

  Mode path: /
  GLUE></code></pre>


          
         Notice from the first line that I'm in the 'Parvoviridae-GLUE' directory when
         I initiate GLUE.
         This means that, by default, it will be my 'working directory' - when I reference
         a file from the GLUE console, I'll need to do so <u>relative to this directory</u>.
 	   </p>

	   <p>
 
         Next, navigate to the 'parvoviridae' project as shown here:
	   </p>

  <pre>
  <code>GLUE> project parvoviridae 
  OK
  Mode path: /project/parvoviridae/</code></pre>

 	   </p>
 
	   <p>
         Now create the module using it's
		 <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/modules/genus/parvo/parvoviridaeNcbiImporterAmdv.xml"><b> configuration file</b></a>,
         which is contained in the Parvoviridae-GLUE project, as shown here:

  <pre>
  <code>Mode path: /project/parvoviridae/
  GLUE> create module -f modules/genus/parvo/parvoviridaeNcbiImporterAmdv.xml
  OK
  (1 Module created) </code></pre>

 	   </p>

 	   <p>
         To run the module, execute the following command in the GLUE shell:

  <pre>
  <code>Mode path: /project/parvoviridae/
  GLUE> module parvoviridaeNcbiImporterAmdv import</code></pre>

 	   </p>
 	   <p>

         When I ran this command on the 1st December 2020, I obtained the following output:


  <pre>
  <code>ncbiImporterSummaryResult
  totalMatching: 1250
  present: 0
  surplus: 0
  missing: 1250
  deleted: 0
  downloaded: 1250</code></pre>

         Since the GenBank database is continually expanding, you may find
         that you obtain more sequences than this when running the same 
         command at a later date. Note that next time the module is run, only 
         the missing sequences will be downloaded. 
	   </p>
 
 	   <p>
 	     Now export the sequences to a 'source' directory as follows.
 	   

   <pre>
   <code>Mode path: /project/parvoviridae
   GLUE> export source ncbi-curated-parvo-amdv -p sources/</code></pre>

		  The sequences are now in your local copy of the Parvoviridae-GLUE project, under
		  './sources/ncbi-curated-parvo'. However, they are in XML format - not
		  particularly convenient for human use - so how do we 
		  get our hands on the FASTA versions of these sequences?
		</p>

 	   <p>
          GLUE is designed to provide as much flexibility as possible with respect 
          to data-formatting. To export FASTA-formatted sequences from GLUE, use a
          <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/moduleReference/moduleType/fastaExporter">'<b>fastaExporter</b>'</a>       
          
          
          module. As with many of GLUE's core module types, a
          version with a default configuration can be created on
          the GLUE command line using the '-t' (template) option:
          
   <pre>
   <code>Mode path: /project/parvoviridae
   GLUE> create module -t fastaExporter 
   OK
   (1 Module created)</code></pre>
		</p>

 	   <p>

          Now lets use this module to export the sequences in the 'ncbi-curated-parvo-amdv'
          source. 
          
  <pre>
  <code>Mode path: /project/parvoviridae
  GLUE> module fastaExporter export -w "source.name = 'ncbi-curated-parvo-amdv'" -f amdv.fna
  OK</code></pre>
          
  	   <p>
          You can check how to use any of the commands associated with a GLUEs module
          using the 'help' option.

 <pre>
  <code>Mode path: /project/parvoviridae
  GLUE> module fastaExporter help export</code></pre>
  
    	   </p>

  
 	   <h3>
		 <a id="ParseGenBankXML" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>2. Importing AMDV sequences and metadata from GenBank files</strong>.
	   </h3>


	   <p>
         Now that we've downloaded a set of AMDV sequences, we can incorporate them into 
         the Parvoviridae-GLUE project.
         Load the sequences as follows:

  <pre>
  <code>Mode path: /project/parvoviridae
  GLUE> import source sources/ncbi-curated-parvo-amdv/</code></pre>
 	   </p>
 	   
	   <p>
          All AMDV sequences obviously have the same taxonomic information.
          To input the taxonomic data for each sequence in the main 'sequence' table,
          we can use
		  <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/glue/genus/parvo/exampleLoadSequenceDataBdfv.js"><b>this script</b></a>.
	      which utilise two GLUE commands: (1) listing all sequences in a particular source;
	      (ii) 
	   </p>
 	   
	   <p>
		  The script is run as follows:

  <pre>
  <code>Mode path: /project/parvoviridae
  GLUE> run script glue/genus/parvo/exampleLoadSequenceDataBdfv.js</code></pre>
 	   
	   <p>
          Link each new AMDV sequence to the isolate table using
          <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/glue/genus/parvo/exampleLoadSequenceDataBdfv.js"><b>this script</b></a>.
  <pre>
  <code>Mode path: /project/parvoviridae
  GLUE> run script glue/genus/parvo/exampleLoadIsolateDataBdfv.js</code></pre>

 	   </p>


	   <p>
         Extracting information from GenBank fields using a
         customised version
         of GLUE's 'genbankPopulator' module.
         We can use this module to search for for <i>host species</i>,
         <i>time and place of isolation</i>,
         and <i>isolate</i> data in the "notes" section of the GenBank XML. 
 	   </p>
 	   
	   <p>
         Use of FreeMarker and REGEX in this and other GLUE modules.
         If unfamiliar the examples included in this project can be modified. 
  
  
  <pre>
  <code>Mode path: /project/parvoviridae
  GLUE> run script glue/genus/parvo/exampleLoadIsolateDataBdfv.js</code></pre>

 	   </p>


		 <p>

           Inspect data in the sequence table via the command line:

  <pre>
  <code>Mode path: /project/parvoviridae
  GLUE> list sequence sequenceID name full_name family genus clade
  +============+===============+================+==================+==============+
  | sequenceID |     name      |     family     |      genus       |    clade     |
  +============+===============+================+==================+==============+
  | AB182568   | AMDV          | Parvoviridae   | Parvovirus       | Avian-1      |
  | AB182569   | AMDV          | Parvoviridae   | Parvovirus       | Avian-1      |
  | AB182570   | AMDV          | Parvoviridae   | Parvovirus       | Avian-1      |
  | AB182571   | AMDV          | Parvoviridae   | Parvovirus       | Avian-1      |
  | AB182572   | AMDV          | Parvoviridae   | Parvovirus       | Avian-1      |</code></pre>

  		 </p>


		 <p>
           Inspect data in the isolate table via the command line. This time using a
          <a target="_blank" href="https://www.w3schools.com/sql/sql_where.asp"><b>'where clause'</b></a>         
           to limit the query to Japanese isolates only:

  <pre>
  <code>Mode path: /project/parvoviridae
  GLUE> list custom-table-row isolate id isolate country -w "country = 'Japan'"
  +==========+============+=========+
  |    id    |  isolate   | country |
  +==========+============+=========+
  | AB277726 | AGP-SA1P-1 | Japan   |
  | AB277727 | AGP-SA1P-2 | Japan   |
  | AB277728 | AGP-SA1P-3 | Japan   |
  | AB277729 | AGP-SA1P-4 | Japan   |
  | AB277730 | AGP-SA1P-5 | Japan   |</code></pre>		   


  		 </p>

		 <p>
           Executing the query above without a where clause will show that
           not every sequence has complete information for each field in the isolate
           table. This is because typically, some GenBank entries for any given virus
           lack this information.
  		 </p>


	   <h3>
		 <a id="FilterDataSet" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>3. Filtering data to obtain a valid analysis data set</strong>.
	   </h3>




		 <p>
           Investigating the data in the sequence table reveals that some of the 
           AMDV sequences are not associated with isolate data. This means that our
           genbankPopulator module didn't find the information in the XML file for that 
           sequence. Such inconsistent annotation of virus entries in GenBank is 
           fairly commonplace, due to a lack of standards.
  		 </p>


 		 <p>
           It's often possible to fill in missing values via reference to an 
           associated manuscript. In this section I'll show how to use GLUE to 
           coordinate this process.
  		 </p>


 		 <p>
           
           First, let's export the existing isolate table data for sequence in our source.
           This is a three step process:


  <pre>
  <code>Mode path: /project/parvoviridae

  GLUE> console set cmd-output-file-format tab 
  OK
  Mode path: /project/parvoviridae
  GLUE> console set next-cmd-output-file amdv-data.tsv
  OK
  Mode path: /project/parvoviridae
  GLUE> list custom-table-row isolate id isolate country host_sci_name collection_year</code></pre>		

		 </p>
		 <p>
           Show format for updating data - (template).
		 </p>

		 <p>
           Use       
		   <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/glue/genus/parvo/exampleImportCuratedSideData.js"><strong>this script</strong></a>
		   to upload the refined data.
		 </p>

		 <p>
		   If you improve metadata for any virus sequence set, you may want to share.
		   See sharing section below.
		 </p>






	   <h3>
		 <a id="RefineReference" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>4. Refining the AMDV reference sequence</strong>.
	   </h3>


		 <p>
		   The 'Parvovirus' extension of Parvoviridae-GLUE includes reference sequences for 
		   most (if not all) species, including AMDV.
		   The reference sequence declarations can be found
		   <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/glue/genus/parvo/parvoReferences.glue"><b>in this file</b></a>.
		 </p>
	   
	   
		 <p>
		   However, in a project focussed specifically on AMDV, we may want to annotate this 
		   specific reference more richly - incorporating everything that is known about
		   the virus genome.
		 </p>

	   
		 <p>
           Let's examine how we can modify the existing AMDV reference sequence.
           First, let's have a look at the reference sequences in the parvoviridae project.
  	   
  		 </p>
  		 
  		 
		 <p>
           As expected, we see the master references of the core project,
           plus the references associated with the Parvovirus extension.
           Navigate to the AMDV reference sequence and list all the features that 
           have been associated with this reference, as follows:

  <pre>
  <code>Mode path: /project/parvoviridae
  GLUE> reference REF_Parvo_AMDV 
  OK
  Mode path: /project/parvoviridae/reference/REF_Parvo_AMDV
  GLUE> list feature-location 
  +========================+==============+
  | referenceSequence.name | feature.name |
  +========================+==============+
  | REF_Parvo_AMDV         | whole_genome |
  | REF_Parvo_AMDV         | Rep68          |
  | REF_Parvo_AMDV         | VP1          |
  +========================+==============+
  FeatureLocations found: 4</code></pre>	        

  		 </p>
  		 
  		 
		 <p>
 		   The reference has the locations of major genome features mapped - specifically the 
		   <i>rep</i> and <i>cap</i> genes.	   
		   Inspecting the <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001944"><b>AMDV GenBank entry</b></a>
		   shows that includes annotation for an additional, hypothetical open reading frame (ORF) called "V2".
  		 </p>
  		 
  		 
		 <p>
           Let's modify the existing AMDV reference ssequence to include V2. 
           First, lets examine all of the features defined in the entire Parvoviridae-GLUE 
           project.

  <pre>
  <code>GLUE> project-mode 
  OK
  Mode path: /project/parvoviridae
  GLUE> list feature
  +==============+==============+===================================+
  |     name     | parent.name  |            description            |
  +==============+==============+===================================+
  | whole_genome | -            | Whole genome                      |
  | 5flank       | -            | 5 prime genomic flanking sequence |
  | 3flank       | -            | 3 prime genomic flanking sequence |
  | Rep68          | whole_genome | Rep68licase protein                 |
  | ORFV2        | whole_genome | Hypothetical protein ORFV2        |
  | ORFC2        | whole_genome | Hypothetical protein ORFC2        |
  | ORF4         | whole_genome | Hypothetical protein ORF4         |
  | ORF3         | whole_genome | Hypothetical protein ORF3         |
  | nonanuc      | whole_genome | nonanucleotide motif              |
  | intergenic   | whole_genome | intergenic region                 |
  | gp3          | whole_genome | gp3 protein                       |
  | gp2          | whole_genome | gp2 protein                       |
  | VP1          | whole_genome | VP1sid protein                    |
  | 5UTR         | whole_genome | 5' UTR                            |
  | 3UTR         | whole_genome | 5' 3TR                            |
  +==============+==============+===================================+
  Features found: 15</code></pre>


  		 </p>

  		 
		 <p>
           First let's create the feature, as follows:
  <pre>
  <code>Mode path: /project/parvoviridae
  GLUE> create feature v2 "v2 gene"
  OK
  (1 Feature created)
  
  GLUE> feature v2 set metatag CODES_AMINO_ACIDS true
  OK
  
  Mode path: /project/parvoviridae/
  GLUE> feature v2 set metatag OWN_CODON_NUMBERING true
  OK</code></pre>

 		 </p>

  		 
		 <p>

           Now lets associate this feature with a location in the AMDV genome:

  <pre>
  <code>Mode path: /project/parvoviridae
  GLUE> reference REF_Parvo_AMDV 
  OK
  Mode path: /project/parvoviridae/reference/REF_Parvo_AMDV</code></pre>	
  		 </p>

		 <p>

           Now we can really get to work! In the next few steps we will use everything
           we've created so far to start doing some proper comparative analysis.
           [Advantages of the work done - ability to preserve it so it doesn't need to
           be repeated].
  		 </p>



        <p><img src="./images/parrots-midlevel.png" alt="Parrots" /></p>


	   <h3>
		 <a id="CreateAlignment" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>5. Creating an alignment of all AMDV sequences</strong>.
	   </h3>

	   
		 <p>
		   A pairwise, reference guided approach is an efficient and reliable way to 
		   align sequences derived from a single virus species
		   The GLUE framework allows this approach to be used in a codon-aware way, and
		   without data loss.
		 </p>

		 <p>

		   At this point, I should mention that some AMDV sequences are in the opposite
		   orientation to our chosen AMDV reference. To address this, run
		   <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/glue/genus/parvo/exampleApplyReverseComplement.glue"><b>this GLUE file</b></a>.
		   This will allow us to proceed with directly to the alignment step with all 
		   sequences in the same orientation. Later in this section,
		   I'll explain how I created the file.


  <pre>
  <code>run file glue/genus/parvo/exampleApplyReverseComplement.glue</code></pre>


		 </p>

	   
		 <p>
		   To create an alignment, first, exit the reference sequence to project level (assuming you are still 
		   in this mode path):
		 </p>

   <pre>
   <code>Mode path: /project/parvoviridae/reference/REF_Parvo_AMDV
   GLUE> project-mode 
   OK
   Mode path: /project/parvoviridae
   GLUE></code></pre>
	   
		 <p>
		   Before we perform the alignment construction, we need to first create 
		   a <b>constrained alignment</b> object and specify which sequences belong in it:

  <pre>
  <code>Mode path: /project/parvoviridae
  GLUE> create alignment AL_AMDV -r REF_Parvo_AMDV
  OK
  Mode path: /project/parvoviridae
  GLUE> alignment AL_AMDV add member --whereClause "source.name = 'ncbi-curated-parvo-amdv'"
  OK</code></pre>
  
		 </p>


		 <p>
		   Now we can construct the alignment.
		   The Parvoviridae-GLUE project contains a 
		   <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/modules/genus/parvo/parvovirusCompoundAligner.xml"><b>parvovirus-specific configuration file</b></a>
		   for GLUE's  'compoundAligner' module.
  <pre>
  <code>compute alignment AL_AMDV parvovirusCompoundAligner</code></pre>
		 </p>

  

		 <p>
		   Now let's inspect the resulting alignment:
  <pre>
  <code>GLUE> alignment AL_AMDV show statistics</code></pre>

		 </p>

		 <p>
		   Or alternatively, list alignment members, along with their associated data, like this:

  <pre>
  <code>GLUE> alignment AL_AMDV 
  OK
  Mode path: /project/parvoviridae/alignment/AL_AMDV
  GLUE> list member sequence.sequenceID sequence.name sequence.isolate.isolate
  +=====================+===============+====================================+
  | sequence.sequenceID | sequence.name |      sequence.isolate.isolate      |
  +=====================+===============+====================================+
  | AB182568            | AMDV          | YCC-IN1P                           |
  | AB182569            | AMDV          | YCC-IN2P                           |
  | AB182570            | AMDV          | RRP-JP11P                          |
  | AB182571            | AMDV          | RRP-JP12P                          |
  | AB182572            | AMDV          | BTP-SA11P                          |</code></pre>
  
 		 </p>

		 <p>
		   (only the first few results are shown here)
		 </p>

	   
		 <p>
		   How did I figure out which ones? There are lots of ways to do this 
		   using GLUE. In this case I used a fairly unsophisticated approach 
		   (but one that seemed to work quite well).
		 </p>

	   
		 <p>
		   I naively aligned all AMDV sequences then inspected the alignment
		   in the console. Since we had excluded sequences shorter than 200bp when we
		   downloaded from AMDV sequences GenBank (see above), it was a reasonable guess that any
		   sequences for which the only aligned segments were drastically shorter than
		   this length were in reverse orientation relative to the reference.
		 </p>

	   
		 <p>
		   
		   Additionally, 
		   since orientation of sequences is usually consistent within a study, it
		   wasn't particularly difficult to create a list of sequences suspected to 
		   require reverse complementation. 
		 </p>

	   
		 <p>
		   Refining the alignment - may want to manually adjust and re-import.
		   Note, the structure of the alignments directory - refer to alignment tree.
		   AMDV is a 'tip' alignment'.
		 </p>

   <pre>
   <code>Mode path: /project/parvoviridae/
   GLUE> import alignment -f alignments/tip/</code></pre>
	   

		 <h3>
		   <a id="MapCoverage" class="anchor" href="#MapCoverage" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>6. Mapping feature coverage in AMDV sequences</strong>.
		 </h3>

	   
		 <p>
		   Now that we have created an alignment of AMDV sequences that is constrained to 
		   the AMDV reference sequence, we can use this alignment in combination with GLUE's
          <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/moduleReference/moduleType/featurePresenceRecorder">'<b>featurePresenceRecorder</b>'</a>       
		   module to examine feature coverage within individual AMDV sequences.
		   Run the module as follows:
		 </p>

  <pre>
  <code>Mode path: /project/parvoviridae
  GLUE> create module -f modules/genus/parvo/parvoFeaturePresenceRecorder.xml 
  OK
  (1 Module created)
  Mode path: /project/parvoviridae
  GLUE> module parvoFeaturePresenceRecorder
  OK
  Mode path: /project/parvoviridae/module/parvoFeaturePresenceRecorder
  GLUE> record feature-presence AL_AMDV -r -f whole_genome -d
  OK
</code></pre>

	   
		 <p>
		   This can also be accomplished by by running
		 <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/glue/genus/parvo/exampleRecordFeaturePresenceAmdv.glue"><b>this file</b></a>,
		 as follows:
		 </p>

	   <h3>
		 <a id="buildPhylogenies" class="anchor" href="#buildPhylogenies" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>7. Reconstructing phylogenetic trees</strong>.
	   </h3>
	   
	   <p>
         In the final step in this tutorial, we will create a set of midpoint-rooted
         tree for each of the two main genes in the AMDV genome.
	   </p>

	   <p>
         First, lets build trees for both VP1 and Rep68. We are going to use GLUE's
          <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/moduleReference/moduleType/raxmlPhylogenyGenerator">'<b>raxmlPhylogenyGenerator</b>'</a>       
         
         
         The entire sequence of commands for creating both trees can be found in
		 <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/glue/genus/parvo/exampleCreateAmdvPhylogenies.glue"><b>this file</b></a>,
		 which is in the Parvoviridae-GLUE repo, as follows:
  <pre>
  <code>Mode path: /project/parvoviridae
  GLUE> run file glue/genus/parvo/exampleCreateAmdvPhylogenies.glue</code></pre>

		 Before you attempt this, bear in mind that we are about to build a bootstrapped,
		 maximum likelihood tree for thousands of sequences. On my computer it took 
		 over an hour to build trees for both genes.  
		 
	   </p>

	   
		<p>
		  To midpoint-root the trees, we will use GLUE's
		  module. To apply the module to the two trees we've constructed, run
		  <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/glue/genus/parvo/exampleRerootPhylogeny.glue"><b>this file</b></a>
		  as follows:


  <pre>
  <code>Mode path: /project/parvoviridae
  GLUE> run file glue/genus/parvo/exampleRerootPhylogeny.glue</code></pre>

         Note that trees could also be midpoint rooted by importing them into another application
         that offers this functionality. However, it is often convenient to be able to 
         do this within GLUE.
         
		</p>
	 
 	   
		<p>
		  Export annotations for the trees by running
		  <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/glue/genus/parvo/exampleExportAnnotations.glue"><b>this file</b></a>,
		  as follows:


  <pre>
  <code>Mode path: /project/parvoviridae
  GLUE> run file glue/genus/parvo/exampleRerootPhylogeny.glue</code></pre>


		</p>	 
  
  
  
		<p>
		  The finished trees with annotations can be found 
		  <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/tree/master/trees/virus-level"><b>here</b></a>.
          One way to view these trees is by using Andrew Rambaut's
		  <a target="_blank" href="https://github.com/rambaut/figtree/releases"><b>FigTree</b></a>
		  program.
		</p>



		<h3>
		  <a id="CollaborativeUse" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Collaboratively developing Parvoviridae-GLUE resources</strong>.
		</h3>

		<p>
		  Comparative analysis hampered by laborious data preparation.
		  GLUE is open source software for dealing with the
		  nuts and bolts of comparative sequence analysis.
		  It has been designed to <b>facilitate</b>.
		 
		</p>
	 
		<p>
		  Host projects on GitHub.
		  Clone. fork.  
		</p>

       <hr>

	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>

		Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		<b>(2018)</b>
		<br>
		GLUE: A flexible software system for virus sequence data.
		<br>
		<i>BMC Bioinformatics</i>
		<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		<br>
		<br>



       <hr>


	   <h3>
		 <a id="Related resources" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related resources</strong>
	   </h3>
 


		 <ul>
		   <li> <b>The history of fur farming</b>:
		   </li>
		   <li> <b>Comparative phylogenetic analysis of viruses</b>:
		   </li>
		   <li> <b>Introduction to the console</b>:
		   </li>
		 </ul>


        


	  <footer class="site-footer">
	 
		<span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/">
		  Parvoviridae-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.
		</span>

		<span class="site-footer-credits">
		  This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.
		</span>
	   
	  </footer>

    </section>


  </body>
  
</html>



