<!DOCTYPE html>
<html lang="en-us">

  <head>

    <meta charset="UTF-8">
    <title>Parvoviridae-GLUE by giffordlabcvr: Paleovirology components</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Parvoviridae-GLUE</h1>
      
      <h2 class="project-tagline">A GLUE project for the Parvoviridae.</h2>
      <a href="../index.html" class="btn">Home</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="./tutorial.html" class="btn">Tutorial</a>

      <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/tree/master" class="btn">View on GitHub</a>
      
    </section>

    <section class="main-content">


	   <h3>
		 <a id="GlueProject" class="anchor" href="#GlueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Comparative genomic analysis of parvoviruses using Parvoviridae-GLUE</strong>
	   </h3>
	   <hr>
	   
	   
	   <p>
         Parvoviridae-GLUE is a data resource designed to facilitate comparative analysis of
         parvovirus genomes. It was created using the <a href="http://glue-tools.cvr.gla.ac.uk/"><b>GLUE</b></a>
         software environment. 
	   </p>


	   <p>

         The Parvoviridae-GLUE project not only contains data items required for
         comparative genomic analysis
         - i.e. virus genome sequences, multiple sequence alignments, genome feature annotations, and other
         sequence-associated data -
         but also, loading the project into the GLUE software framework
         creates a relational database that represents the semantic relationships
         between these data items. This creates powerful framework for performing
         <i>standardised</i> comparative analysis, via GLUE's command layer.
	   </p>





		 
       <br>
	   <h3>
		 <a id="sequenceData" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Sequence data</strong>
	   </h3>
       <hr>


	   <p>
		 The sequence data in this project derive from multiple distinct sources:<br>
	   </p>
	   	  	   <p>	      
		 <b>ncbi-refseqs</b>: Genome-length reference sequences of <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/core/cress-refseqs-side-data.tsv">representative CRESS DNA virus species</a>.		 
		 These
		 <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/tree/master/sources/ncbi-refseqs">XML-formatted files</a>
		  are downloaded directly from <a href="https://www.ncbi.nlm.nih.gov/genbank/">NCBI</a>
		 and are uniquely identified within this project by their <a target="_blank" href="https://www.ncbi.nlm.nih.gov/Sequin/acc.html">GenBank accession numbers</a>. 
	   </p>
	   
	   <p>
		 <b>fasta-curated</b>: A non-redundant set of 
		 <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/locus/ecv-locus-data.tsv">CRESS DNA virus-derived EVE loci</a>.
		 These
		 <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/tree/master/sources/fasta-curated-ecv">FASTA sequences</a>
		 have been curated via systematic screening of whole genome sequence (WGS) assemblies using the 
		 <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">DIGS tool</a>.
		 Sequences in this source have unique IDs based on arbitrary numbering. 
	   </p>
	      	   
	   <p>
		 <b>fasta-refseqs</b>:
		 
		 <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/eve/ecv-refseqs-side-data.tsv">EVE reference sequences</a>
		 - i.e. best-guess estimates of the ancestral sequences of
		 the CRESS DNA viruses that gave rise to EVEs.
		 Where possible these are consensus/ancestral sequences derived from alignments
		 <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/tree/master/alignments/eve/tip/">included in this project</a>.
		 Sequence IDs used in this source correspond to the names of the unique EVE loci they represent
		 (see <a href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1" target="_blank">here</a> for details).
	   </p>
 
		 
       <br>
	   <h3>
	   <a id="sequenceData" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Sequence-associated data</strong>
	   </h3>		 
       <hr>
	   
	   <p>
         Sequences included in this project are linked to auxiliary data in tabular format, this includes:
		 
		 <ol>
		   <li><a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/eve/ecv-refseqs-side-data.tsv">Basic taxonomic data</a> for genome-length virus reference sequence in <b>ncbi-refseqs</b>.</li> 
		   <li><a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/locus/ecv-locus-data.tsv">Locus data</a> for the EVE sequences in <b>fasta-curated</b>.</li> 
		 </ol>
		 
	   </p>
		 
       <br>
	   <h3>
		 <a id="alignments" class="anchor" href="#alignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments (MSAs)</strong>
	   </h3>
       <hr>
	   
	   <p>
		 Several distinct categories of MSA are included in this project, each representing a distinct taxonomic level.
		
		 <ol>
		   <li><b>Tip (i)</b>: Virus species (genome-length alignment)</li> 
		   <li><b>Tip (ii)</b>: <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/tree/master/alignments/eve/tip/">EVE lineages</a>.
		   These alignments contain sets of EVE sequences derived from the same ancestral germline colonisation event (i.e. orthologs or duplicates)</li> 
		   <li><b>Internal</b>: <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/tree/master/alignments/eve/internal">Virus genera</a></li> 
		   <li><b>Root</b>: <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/tree/master/alignments/virus/root">Viruses and EVE reference sequences</a></li> 
		 </ol>
	   </p>





	   <br>
	   <h3>
		 <a id="virusReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Parvovirus reference sequences</strong>
	   </h3>
	   <hr>



		 <p>
		 Three subfamilies are currently recognised within the family:
		 
		 <ol>
		   <li><a href="">Parvovirinae</a> </li>
		   <li><a href="">Epiparvovirinae</a> </li> 
		   <li><a href="">Densovirinae</a> </li> 
		 </ol>



		  We used GLUE to derive annotated reference sequences for all known parvovirus species.
		  Reference sequences are linked to auxiliary data in		 
		  <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/tabular/core/parvo-ncbi-refseqs-side-data.tsv"><b>tabular format</b></a>.
		</p>
		
		<p>

 		  For each parvoviral genus, we defined a 'master' reference sequence, as follows:
		  <ul>
		    <li><b><i>Protoparvovirus</i></b>: Hepatitis B virus, strain ayw (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_003977"><b>NC_003977</b></a>)</li>
		    <li><b><i>Aviparvovirus</i></b>: Duck hepatitis B virus, isolate DHBVQCA34 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001344"><b>NC_001344</b></a>)</li>
		    <li><b><i>Herpetoparvovirus</i></b>: Tibetan frog hepatitis B virus, isolate 243398 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_030446"><b>NC_030446</b></a>)</li>
		    <li><b><i>Metaparvovirus</i></b>: Bluegill hepatitis B virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_030445"><b>NC_030445</b></a>)</li>
		    <li><b><i>Paraparvovirus</i></b>: White sucker parvovirus, isolate RR173 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_027922"><b>NC_027922</b></a>)</li>
		  </ul>

		</p>
		
		<p>

          We explicitly defined the locations of genome features
          on master reference sequences
          (<b>see <a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/glue/core/parvoviridaeExplicitFeatureLocations.glue">here</b></a>).
		  
 		</p>
        


	   <br>
	   <h3>
		 <a id="Multiple sequence alignments" class="anchor" href="#MultipleSequenceAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments</strong>
	   </h3>
	   <hr>


		<p>
          Multiple sequence alignment constructed in this study are linked together using
          GLUE's 
          ‘<a href="./alignmentTree.html">alignment tree</a>’
          data structure. Alignments in the project include:
         
		  
		  <ol>
			<li> A ‘<a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/blob/master/alignments/root/parvoviridae.aln.fna"><strong>root</strong></a>’
			alignment constructed to represent proposed
			homologies between representative members of major parvovirus lineages</li>
			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/tree/master/alignments/internal/"><strong>genus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific parvovirus genera.</li>
		  </ol>
         
 		</p>



	   <br>
	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees</strong>
	   </h3>
	   <hr>


		<p>

          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
        
		  <ol>
			<li> Recursively populated <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/tree/master/trees/main-root"><strong>root</strong></a> phylogeny (Rep)</li>
			<li> <a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/tree/master/trees/main-genus"><strong>Genus-level</strong></a> phylogenies</li>
		  </ol>

 		</p>






	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Parvoviridae-GLUE/">Parvoviridae-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



